<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
<!-- 2022-03-07 Mon 16:36 -->
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<title>&lrm;</title>
<meta name="generator" content="Org mode" />
<meta name="author" content="Kevin Dudley" />
<style type="text/css">
 <!--/*--><![CDATA[/*><!--*/
  .title  { text-align: center;
             margin-bottom: .2em; }
  .subtitle { text-align: center;
              font-size: medium;
              font-weight: bold;
              margin-top:0; }
  .todo   { font-family: monospace; color: red; }
  .done   { font-family: monospace; color: green; }
  .priority { font-family: monospace; color: orange; }
  .tag    { background-color: #eee; font-family: monospace;
            padding: 2px; font-size: 80%; font-weight: normal; }
  .timestamp { color: #bebebe; }
  .timestamp-kwd { color: #5f9ea0; }
  .org-right  { margin-left: auto; margin-right: 0px;  text-align: right; }
  .org-left   { margin-left: 0px;  margin-right: auto; text-align: left; }
  .org-center { margin-left: auto; margin-right: auto; text-align: center; }
  .underline { text-decoration: underline; }
  #postamble p, #preamble p { font-size: 90%; margin: .2em; }
  p.verse { margin-left: 3%; }
  pre {
    border: 1px solid #ccc;
    box-shadow: 3px 3px 3px #eee;
    padding: 8pt;
    font-family: monospace;
    overflow: auto;
    margin: 1.2em;
  }
  pre.src {
    position: relative;
    overflow: auto;
    padding-top: 1.2em;
  }
  pre.src:before {
    display: none;
    position: absolute;
    background-color: white;
    top: -10px;
    right: 10px;
    padding: 3px;
    border: 1px solid black;
  }
  pre.src:hover:before { display: inline; margin-top: 14px;}
  /* Languages per Org manual */
  pre.src-asymptote:before { content: 'Asymptote'; }
  pre.src-awk:before { content: 'Awk'; }
  pre.src-C:before { content: 'C'; }
  /* pre.src-C++ doesn't work in CSS */
  pre.src-clojure:before { content: 'Clojure'; }
  pre.src-css:before { content: 'CSS'; }
  pre.src-D:before { content: 'D'; }
  pre.src-ditaa:before { content: 'ditaa'; }
  pre.src-dot:before { content: 'Graphviz'; }
  pre.src-calc:before { content: 'Emacs Calc'; }
  pre.src-emacs-lisp:before { content: 'Emacs Lisp'; }
  pre.src-fortran:before { content: 'Fortran'; }
  pre.src-gnuplot:before { content: 'gnuplot'; }
  pre.src-haskell:before { content: 'Haskell'; }
  pre.src-hledger:before { content: 'hledger'; }
  pre.src-java:before { content: 'Java'; }
  pre.src-js:before { content: 'Javascript'; }
  pre.src-latex:before { content: 'LaTeX'; }
  pre.src-ledger:before { content: 'Ledger'; }
  pre.src-lisp:before { content: 'Lisp'; }
  pre.src-lilypond:before { content: 'Lilypond'; }
  pre.src-lua:before { content: 'Lua'; }
  pre.src-matlab:before { content: 'MATLAB'; }
  pre.src-mscgen:before { content: 'Mscgen'; }
  pre.src-ocaml:before { content: 'Objective Caml'; }
  pre.src-octave:before { content: 'Octave'; }
  pre.src-org:before { content: 'Org mode'; }
  pre.src-oz:before { content: 'OZ'; }
  pre.src-plantuml:before { content: 'Plantuml'; }
  pre.src-processing:before { content: 'Processing.js'; }
  pre.src-python:before { content: 'Python'; }
  pre.src-R:before { content: 'R'; }
  pre.src-ruby:before { content: 'Ruby'; }
  pre.src-sass:before { content: 'Sass'; }
  pre.src-scheme:before { content: 'Scheme'; }
  pre.src-screen:before { content: 'Gnu Screen'; }
  pre.src-sed:before { content: 'Sed'; }
  pre.src-sh:before { content: 'shell'; }
  pre.src-sql:before { content: 'SQL'; }
  pre.src-sqlite:before { content: 'SQLite'; }
  /* additional languages in org.el's org-babel-load-languages alist */
  pre.src-forth:before { content: 'Forth'; }
  pre.src-io:before { content: 'IO'; }
  pre.src-J:before { content: 'J'; }
  pre.src-makefile:before { content: 'Makefile'; }
  pre.src-maxima:before { content: 'Maxima'; }
  pre.src-perl:before { content: 'Perl'; }
  pre.src-picolisp:before { content: 'Pico Lisp'; }
  pre.src-scala:before { content: 'Scala'; }
  pre.src-shell:before { content: 'Shell Script'; }
  pre.src-ebnf2ps:before { content: 'ebfn2ps'; }
  /* additional language identifiers per "defun org-babel-execute"
       in ob-*.el */
  pre.src-cpp:before  { content: 'C++'; }
  pre.src-abc:before  { content: 'ABC'; }
  pre.src-coq:before  { content: 'Coq'; }
  pre.src-groovy:before  { content: 'Groovy'; }
  /* additional language identifiers from org-babel-shell-names in
     ob-shell.el: ob-shell is the only babel language using a lambda to put
     the execution function name together. */
  pre.src-bash:before  { content: 'bash'; }
  pre.src-csh:before  { content: 'csh'; }
  pre.src-ash:before  { content: 'ash'; }
  pre.src-dash:before  { content: 'dash'; }
  pre.src-ksh:before  { content: 'ksh'; }
  pre.src-mksh:before  { content: 'mksh'; }
  pre.src-posh:before  { content: 'posh'; }
  /* Additional Emacs modes also supported by the LaTeX listings package */
  pre.src-ada:before { content: 'Ada'; }
  pre.src-asm:before { content: 'Assembler'; }
  pre.src-caml:before { content: 'Caml'; }
  pre.src-delphi:before { content: 'Delphi'; }
  pre.src-html:before { content: 'HTML'; }
  pre.src-idl:before { content: 'IDL'; }
  pre.src-mercury:before { content: 'Mercury'; }
  pre.src-metapost:before { content: 'MetaPost'; }
  pre.src-modula-2:before { content: 'Modula-2'; }
  pre.src-pascal:before { content: 'Pascal'; }
  pre.src-ps:before { content: 'PostScript'; }
  pre.src-prolog:before { content: 'Prolog'; }
  pre.src-simula:before { content: 'Simula'; }
  pre.src-tcl:before { content: 'tcl'; }
  pre.src-tex:before { content: 'TeX'; }
  pre.src-plain-tex:before { content: 'Plain TeX'; }
  pre.src-verilog:before { content: 'Verilog'; }
  pre.src-vhdl:before { content: 'VHDL'; }
  pre.src-xml:before { content: 'XML'; }
  pre.src-nxml:before { content: 'XML'; }
  /* add a generic configuration mode; LaTeX export needs an additional
     (add-to-list 'org-latex-listings-langs '(conf " ")) in .emacs */
  pre.src-conf:before { content: 'Configuration File'; }

  table { border-collapse:collapse; }
  caption.t-above { caption-side: top; }
  caption.t-bottom { caption-side: bottom; }
  td, th { vertical-align:top;  }
  th.org-right  { text-align: center;  }
  th.org-left   { text-align: center;   }
  th.org-center { text-align: center; }
  td.org-right  { text-align: right;  }
  td.org-left   { text-align: left;   }
  td.org-center { text-align: center; }
  dt { font-weight: bold; }
  .footpara { display: inline; }
  .footdef  { margin-bottom: 1em; }
  .figure { padding: 1em; }
  .figure p { text-align: center; }
  .equation-container {
    display: table;
    text-align: center;
    width: 100%;
  }
  .equation {
    vertical-align: middle;
  }
  .equation-label {
    display: table-cell;
    text-align: right;
    vertical-align: middle;
  }
  .inlinetask {
    padding: 10px;
    border: 2px solid gray;
    margin: 10px;
    background: #ffffcc;
  }
  #org-div-home-and-up
   { text-align: right; font-size: 70%; white-space: nowrap; }
  textarea { overflow-x: auto; }
  .linenr { font-size: smaller }
  .code-highlighted { background-color: #ffff00; }
  .org-info-js_info-navigation { border-style: none; }
  #org-info-js_console-label
    { font-size: 10px; font-weight: bold; white-space: nowrap; }
  .org-info-js_search-highlight
    { background-color: #ffff00; color: #000000; font-weight: bold; }
  .org-svg { width: 90%; }
  /*]]>*/-->
</style>
<link rel="stylesheet" href="tufte.css" type="text/css" />
<script type="text/javascript">
// @license magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt Public Domain
<!--/*--><![CDATA[/*><!--*/
     function CodeHighlightOn(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.add("code-highlighted");
         target.classList.add("code-highlighted");
       }
     }
     function CodeHighlightOff(elem, id)
     {
       var target = document.getElementById(id);
       if(null != target) {
         elem.classList.remove("code-highlighted");
         target.classList.remove("code-highlighted");
       }
     }
    /*]]>*///-->
// @license-end
</script>
</head>
<body>
<div id="content">
<div id="table-of-contents">
<h2>Table of Contents</h2>
<div id="text-table-of-contents">
<ul>
<li><a href="#org44c7f6b">1. <b>Structure and dynamics of the bacterial communities colonising low- and high-quality fodder in the bovine rumen</b></a></li>
<li><a href="#org70f0865">2. <b>Abstract</b></a></li>
<li><a href="#orgf39ff22">3. <b>1. Introduction</b></a></li>
<li><a href="#org29c1de1">4. <b>2. Materials and methods</b></a>
<ul>
<li><a href="#org71f0d07">4.1. <b>2.1 Materials</b></a></li>
<li><a href="#orgdde9662">4.2. <b>2.2 Fodder compositional analysis</b> </a></li>
<li><a href="#org7f2f740">4.3. <b>2.3 <i>In situ</i> incubation of fodder in the bovine rumen</b></a></li>
<li><a href="#org0c9d980">4.4. <b>2.4 Fodder analysis post-digestion</b></a></li>
<li><a href="#org1ed82a6">4.5. <b>2.5 DNA extraction</b></a></li>
<li><a href="#orga8247e9">4.6. <b>2.6 Full-length 16S rRNA amplification and PacBio sequencing</b></a></li>
<li><a href="#orgc7bd8f4">4.7. <b>2.7 Bioinformatics and statistical analyses</b></a></li>
</ul>
</li>
<li><a href="#org07d8dbd">5. <b>3. Results</b></a>
<ul>
<li><a href="#org6cde82e">5.1. <b>3.1 Fodder composition</b></a></li>
<li><a href="#org83f2fa0">5.2. <b>3.2 Degradation of sugarcane bagasse and Lucerne hay in the bovine rumen</b></a></li>
<li><a href="#org44563d4">5.3. <b>3.2 Temporal changes in the bacterial community during bovine rumen degradation of sugarcane bagasse and Lucerne hay</b> </a>
<ul>
<li><a href="#org5bd827d">5.3.1. Epiphytic bacterial communities on sugarcane bagasse and Lucerne hay</a></li>
<li><a href="#org7dfb0d5">5.3.2. Dynamics of changes in the bacterial community adherent to sugarcane bagasse and Lucerne hay during bovine rumen incubation</a></li>
<li><a href="#org076bd47">5.3.3. Comparison of bacterial communities adherent to sugarcane bagasse and Lucerne hay </a></li>
<li><a href="#org3e6a57e">5.3.4. Correlation between degradation of sugarcane bagasse and Lucerne hay and bacterial colonisation during <i>in situ</i> incubation </a></li>
<li><a href="#org306b666">5.3.5. Dynamics of changes in the planktonic bacterial communities </a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#orgaab9a21">6. <b>4. Discussion</b></a></li>
<li><a href="#org8469557">7. <b>5. Conclusion</b></a></li>
<li><a href="#org3acb8f3">8. <b>Acknowledgements</b></a></li>
<li><a href="#orgecb18d7">9. <b>Tables</b></a></li>
<li><a href="#org0a0a8f9">10. <b>Figures</b></a>
<ul>
<li><a href="#org9ecb625">10.1. Figure 1</a></li>
<li><a href="#orgfc3f850">10.2. Figure 2</a></li>
<li><a href="#orgbf7c5f4">10.3. Figure 3</a></li>
<li><a href="#org569925e">10.4. Figure 4</a></li>
<li><a href="#org7b637b9">10.5. Figure 5</a></li>
<li><a href="#org5329117">10.6. Figure 6</a></li>
<li><a href="#org23c9ae9">10.7. Figure 7.</a></li>
</ul>
</li>
<li><a href="#orge2966b0">11. <b>Supplementary figures</b></a></li>
<li><a href="#org26b4db1">12. Figure Legends</a>
<ul>
<li><a href="#org43bde79">12.1. Figure 1. DM (%), NDF (%), and ADF (%) digestibility of sugarcane</a></li>
<li><a href="#orgc8926e3">12.2. Figure 2. The alpha diversity of (A) overall bacterial community</a></li>
<li><a href="#org3611327">12.3. Figure 3. Principal coordinates analysis (PCoA) plot of the beta</a></li>
<li><a href="#org81d6345">12.4. Figure 4. Heatmap of relative abundance of bacterial phyla adherent to</a></li>
<li><a href="#org69cb1c2">12.5. Figure 5. Heatmap of relative abundance of bacterial genera adherent to</a></li>
<li><a href="#org0d3a20a">12.6. Figure 6. Heatmap of the relative distribution of bacterial genera in</a></li>
<li><a href="#orgd078eff">12.7. Figure 7. Heatmap of relative abundance of planktonic bacterial genera</a></li>
<li><a href="#orgf18a509">12.8. Figure S1. The correlation between alpha diversity of bacterial</a></li>
<li><a href="#org7069e9b">12.9. Figure S2. A principal coordinates analysis (PCoA) plot of the beta</a></li>
<li><a href="#org0a8ae2f">12.10. Figure S3. The epiphytic bacterial community adherent to sugarcane</a></li>
<li><a href="#org5558e3d">12.11. Figure S4. Statistically significant differences in the bacterial (A)</a></li>
<li><a href="#org1237ced">12.12. Figure S5. Heatmap of relative abundance of planktonic bacterial phyla</a></li>
</ul>
</li>
<li><a href="#orgf8dae60">13. References</a></li>
<li><a href="#orgd337abb">14. KJD Comments</a>
<ul>
<li><a href="#orgf0612e2">14.1. Corrections</a>
<ul>
<li><a href="#orgeee1fc7">14.1.1. Kevin Dudley ^{4}</a></li>
<li><a href="#org4e39cb5">14.1.2. 60% to 70%</a></li>
<li><a href="#org7a1332d">14.1.3. stomach compartment in</a></li>
<li><a href="#org5402742">14.1.4. DM</a></li>
<li><a href="#org515da96">14.1.5. Microbial colonisation of fodder particles in the rumen</a></li>
<li><a href="#orgd62f450">14.1.6. 72 h of incubation</a></li>
<li><a href="#org1e72437">14.1.7. data not shown in</a></li>
<li><a href="#org8a09982">14.1.8. The proportion of Fibrobacteres was significantly higher in sugarcane bagasse than in Lucerne hay samples at 24 h of incubation, and Lucerne hay had a higher abundance of Tenericutes at 8, 12, and 18 h of incubation than sugarcane bagasse</a></li>
</ul>
</li>
<li><a href="#orga210deb">14.2. Comments</a></li>
</ul>
</li>
</ul>
</div>
</div>

<div id="outline-container-org44c7f6b" class="outline-2">
<h2 id="org44c7f6b"><span class="section-number-2">1</span> <b>Structure and dynamics of the bacterial communities colonising low- and high-quality fodder in the bovine rumen</b></h2>
<div class="outline-text-2" id="text-1">
<p>
Mahsa Abbasabadi ^{1,4}, Paul Whatmore ^{2}, Victoria Coyne ^{3}, Roberto Barrero Gumiel ^{2}, Junior Te'o ^{4}, Ian O'Hara ^{1,5}, <a href="#orgeee1fc7">Kevin J. Dudley ^{3,4}</a>, Karen Harper ^{6}, and Mark D. Harrison ^{1,4*}
</p>

<p>
^{1} Centre for Agriculture and the Bioeconomy, Faculty of Science,
Queensland University of Technology, 2 George Street, Brisbane,
Queensland 4001, Australia
</p>

<p>
^{2} Division of Research and Innovation, Queensland University of
Technology, 2 George Street, Brisbane, Queensland 4000, Australia
</p>

<p>
^{3} Central Analytical Research Facility, Research Infrastructure,
Queensland University of Technology, 2 George Street, Brisbane,
Queensland 4001, Australia
</p>

<p>
^{4} School of Biology and Environmental Sciences, Faculty of Science,
Queensland University of Technology, 2 George Street, Brisbane,
Queensland 4000, Australia
</p>

<p>
^{5} School of Mechanical, Medical and Process Engineering, Faculty of
Engineering, Queensland University of Technology, 2 George Street,
Brisbane, Queensland 4000, Australia
</p>

<p>
^{6} School of Agriculture and Food Science, University of Queensland,
Gatton Campus, Lawes, Queensland 4343, Australia
</p>

<p>
^{<b>}Corresponding author: Mark D. Harrison,
<a href="mailto:md.harrison@qut.edu.au">md.harrison@qut.edu.au</a></b><br />
</p>
<p>
*
</p>
</div>
</div>

<div id="outline-container-org70f0865" class="outline-2">
<h2 id="org70f0865"><span class="section-number-2">2</span> <b>Abstract</b></h2>
<div class="outline-text-2" id="text-2">
<p>
Fodders with different quality may affect the composition of the
microbial communities that colonise and degrade fodder particles in the
rumen. In this study, low-quality fodder (sugarcane bagasse) and
high-quality fodder (Lucerne hay) were incubated in the bovine rumen and
the kinetics of dry matter, neutral detergent fibre, and acid detergent
fibre digestion were determined. Further, the structure of the
planktonic bacterial community and the sessile bacterial communities
colonising sugarcane bagasse and Lucerne hay were characterised using
full-length 16S rRNA PacBio sequencing. Digestion of sugarcane bagasse
and Lucerne hay occurred in three (rapid, intermediate, and slow
digestion) and four (solubilisation, rapid digestion, slow digestion,
and plateau) phases, respectively, in the bovine rumen and each phase
was characterised by distinct sessile bacterial communities. The
relatively rapid phase of sugarcane bagasse digestion was accompanied by
bacteria from the <i>Christensenellaceae R-7 group</i>, <i>Prevotella 1</i>,
<i>Ruminococcus 1</i>, <i>Rikenellaceae RC9</i> gut group, and <i>Lachnospiraceae
XPB1014 group</i>, genera. The intermediate digestion phase of sugarcane
bagasse was accompanied by a shift in bacterial taxa in which
<i>Rikenellaceae RC9</i> gut group, <i>Fibrobacter</i>, <i>Saccharofermentans</i>, and
<i>Treponema 2</i> were enriched. The slow digestion phase in which the rate
of NDF digestion decreased significantly was accompanied by decrease in
abundance of <i>Fibrobacter</i>, <i>Prevotella 1</i>, and <i>Ruminococcus 1</i>.
Digestion of Lucerne hay occurred in three phases: (i) a solubilisation
phase (4 h) in which <i>Christensenellaceae R-7 group</i>, <i>Butyrivibrio 2</i>,
<i>Lachnospiraceae XPB1014 group</i>, and <i>Prevotella 1</i> were the most
abundant bacteria, (ii) a rapid digestion phase (8 &#x2013; 12 h) during which
relative abundance of <i>Treponema 2</i> and <i>Anaeroplasma</i> increased, (iii)
a slow digestion phase (12 - 32 h) in which <i>Rikenellaceae RC9</i> gut
group, <i>Saccharofermentans</i>, and <i>Fibrobacter</i> enriched, and (iv) a
plateau phase (&gt;32 h) in which <i>Ruminococcus 1</i>, <i>Lachnospiraceae
XPB1014 group</i>, and decreased significantly in abundance, and A
comparison of bacterial communities adherent to sugarcane bagasse and
Lucerne hay revealed that the abundance of <i>Fibrobacter</i>,
<i>Saccharofermentans</i>, <i>Butyrivibrio 2</i>, and <i>Anaeroplasma</i> differed
significantly between fodders. This study demonstrated a strong
interaction between the digestion of sugarcane bagasse and Lucerne hay,
and the composition of the fodder-adherent bacterial community. This
study also demonstrated that planktonic bacterial communities did not
change significantly during the period.
</p>

<p>
Keywords: colonisation, full-length 16S rRNA PacBio sequencing,
microbiome, bacterial community, fibre digestion, low-quality fodder
</p>
</div>
</div>

<div id="outline-container-orgf39ff22" class="outline-2">
<h2 id="orgf39ff22"><span class="section-number-2">3</span> <b>1. Introduction</b></h2>
<div class="outline-text-2" id="text-3">
<p>
The global demand for animal products is projected to increase by <a href="#org4e39cb5">60-70%</a> 
in 2050 due to human population growth; hence, additional fodder
will be required to meet the projected increased demand for animal
products. However, fodder production is challenged by food-feed
competition, land degradation, and climate change. In addition, fodder
ingredients with high protein and soluble carbohydrate content are
relatively expensive and compete directly or indirectly with food
production (Ahmadi et al. 2016, Halmemies-Beauchet-Filleau et al. 2018).
Fodder can represent up to 70% of cattle production costs and the
identification of more cost-effective fodder ingredients is timely
because of both market and environmental factors (Ahmadi et al. 2016,
Halmemies-Beauchet-Filleau et al. 2018, McGrath et al. 2018). Thus,
low-quality fodders, including crop processing by-products (<i>e.g.</i>,
sugarcane bagasse), will become increasingly important in the animal
production sector; but there is limited information describing the
direct comparison of digestion and colonisation of low- and high-quality
fodders in the rumen.
</p>

<p>
The rumen is the first and largest of the <a href="#org7a1332d">stomach compartments in</a>
ruminants, and harbours a highly diverse, complex community of obligate
anaerobic bacteria, fungi, protozoa, and archaea. Rumen microorganisms
work synergistically to break down and convert fodder particles into
volatile fatty acids and microbial biomass, key sources of energy and
protein for ruminants, respectively (Mizrahi et al. 2021). The microbial
community in the rumen enables efficient fodder digestion and strongly
influences fodder intake in ruminants. Bacteria are the dominant
microorganisms in the rumen microbial community and make the largest
contribution to the breakdown of fodder particles (Moraïs and Mizrahi
2019). Fodder particle-associated bacteria account for 70 &#x2014; 80% of the
total ruminal bacteria population and produce most of the enzymes for
the digestion of fodders in ruminants (Liu et al. 2016). The attachment
and colonisation of rumen bacteria to fodder particles is essential for
the establishment of the complex microbial community required for their
breakdown (Liu et al. 2016). <a href="#orgfeadd50">Therefore, understanding the mechanism of bacterial attachment and the relationship between fodder digestion and colonisation by rumen bacteria is the stepping stone for the development of methods to increase fodder use efficiency in ruminants through adding feed additives such as enzymes and probiotics and thermochemical pretreatment of low-quality fodders (Mizrahi and Jami 2018, Du et al.2019, Moraïs and Mizrahi 2019, Firkins 2021).</a> <label for="1" class="margin-toggle sidenote-number"></label><input type="checkbox" id="1" class="margin-toggle"/><span class="sidenote">
Can this sentence be simplified?
</span>
</p>

<p>
<a href="#org515da96">It was previously shown that microbial colonisation of fodder particles in the rumen</a> involves three
main steps; (i) displacement of the epiphytic microbial community by
rumen microorganisms (&lt;1 h incubation), (ii) establishment of a primary
colonising community of generalist microorganisms that metabolise
accessible carbohydrates (1 &#x2013; 4 h incubation), and (iii) formation of a
secondary colonising community of specialist microorganisms that digest
cellulose and hemicellulose (&gt;4 h incubation) (Terry et al. 2020). The
composition of the bacterial communities colonising high and low-quality
fodders, including alfalfa hay, barley, rice, and wheat straws and corn
stover, during <i>in situ</i> incubation changes significantly over time and
fodder digestion is dependent on the establishment of primary- and
secondary-colonising microbial communities (Liu et al. 2016, Cheng et
al. 2017, Jin et al. 2018, Terry et al. 2020). While the temporal
changes in the bacterial communities adherent to fodder particles during
incubation in the rumen is well understood, information about changes in
the composition of microbial communities colonising fodder particles of
fodder types with different quality and its correlation with changes in
the chemical composition of fodder during incubation in the rumen is
still limited. Differences in the composition of microbial communities
colonising fodders with different quality can cause shifts in the
microbial metabolic networks which ultimately can affect the <a href="#org5402742">Dry matter (DM)</a>
digestibility, rate of short-chain fatty acids and methane production,
feed intake, and daily body weight gain in ruminants (Cammack et al.
2018, Snelling et al. 2019).
</p>

<p>
Maintaining the productivity of ruminant livestock depends upon factors
that influence the rumen microbiome, including fodder type, composition,
quality, and quantity (Lengowski et al. 2016). Fodder type is well-known
as the principal factor affecting rumination time, <a href="#orgca2744e">rumen fermentation</a><label for="2" class="margin-toggle sidenote-number"></label><input type="checkbox" id="2" class="margin-toggle"/><span class="sidenote">
Is rumen fermentation the correct terminology?
</span>,
and ruminant performance and productivity (Liu et al. 2016). Bagasse,
the main sugar industry by-product, is the lignocellulosic fibre, mostly
composed of cellulose, hemicellulose, and lignin, that remains after the
extraction of sugar juice from sugarcane stalk and has low soluble sugar
and protein contents (Harrison 2016). Despite the relative abundance
(~10 Mt p.a. in Australia) and relatively low cost of this low-quality
fodder, sugarcane bagasse is only used as an emergency livestock feed.
In contrast, Lucerne hay is a high-quality fodder that is used worldwide
as a component of the ruminant livestock diet (Liu et al. 2016).
Sugarcane bagasse and Lucerne hay are known to differ in chemical
composition, physical structure, and subsequently DM digestibility in
the rumen (Pinos-Rodríguez et al. 2002, Harrison 2016, Liu et al. 2016,
So et al. 2020). The DM digestibility of dried and ground sugarcane
bagasse and Lucerne hay were 30% and 73%, respectively after 48 h of
incubation in the rumen (Vitti et al. 1999, Liu et al. 2016). However,
there is limited information about the differences in the bacterial
community that are involved in the breakdown and colonisation of the
fibrous component of sugarcane bagasse and Lucerne hay. In addition,
most of the previous studies have only investigated the effect of
incubation time on the composition of bacterial communities colonising
fodder particles.
</p>

<p>
The kinetics of digestion and composition of the bacterial community
adherent to rice straw and alfalfa hay during <i>in situ</i> incubation in
nonlactating Holstein cows have been characterised and compared (Liu et
al. 2016). Alfalfa hay had significantly higher DM disappearance and NDF
digestibility compared to rice straw during <i>in situ</i> incubation in the
rumen. After 2 and 6 h of incubation in the rumen, alfalfa hay had a
significantly higher relative abundance of <i>Butyrivibrio</i>, <i>Prevotella</i>,
while bacteria from the genera unclassified/ Rikenellaceae/,
unclassified/ Bacteroidales/, and unclassified /Ruminococcaceae/ were
more abundant in the rice straw samples. However, after 16 and 48 h of
incubation, the relative abundance of <i>Fibrobacter</i> was significantly
higher on rice straw, while <i>Anaeroplasma</i> and unclassified
<i>Bacteroidales</i> were more abundant on the surface of alfalfa hay
relative to rice straw (Liu et al. 2016).
</p>

<p>
In this study, we digested sugarcane bagasse and Lucerne hay in the
bovine rumen for 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h, and
measured dry matter, neutral detergent fibre, and acid detergent fibre
digestion. Full-length 16S PacBio sequencing was used to characterise
the bacterial community on the surface of the fodder particles and in
the planktonic bacterial community at each timepoint. This approach
enabled us to compare temporal changes in the bacterial communities on
fodder particles with very different compositions and nutritional
content in a common environment. Further, we were able to investigate
temporal relationships between the planktonic bacterial community and
the bacterial community adherent to the fodder particles during
digestion. Understanding the relationship between the structural and
chemical composition of fodder, the kinetics of fodder digestion, and
bacterial colonisation in the rumen is central to the development of
strategies to enhance the digestibility of sugarcane bagasse.
</p>
</div>
</div>

<div id="outline-container-org29c1de1" class="outline-2">
<h2 id="org29c1de1"><span class="section-number-2">4</span> <b>2. Materials and methods</b></h2>
<div class="outline-text-2" id="text-4">
</div>
<div id="outline-container-org71f0d07" class="outline-3">
<h3 id="org71f0d07"><span class="section-number-3">4.1</span> <b>2.1 Materials</b></h3>
<div class="outline-text-3" id="text-4-1">
<p>
Sugarcane bagasse was obtained directly from the final mill in the mill
train at Racecourse Sugar Mill (Mackay, Australia). Commercial,
high-quality Lucerne hay was provided by Feed Central (Toowoomba,
Australia). Chemicals and consumables for NDF and ADF analyses were
purchased from ANKOM Technology Corp. (USA), including F57 filter bags,
α-amylase, neutral detergent solution concentrate, and triethylene
glycol. Hexadecyltrimethylammonium bromide (≥ 99%) and anhydrous sodium
sulphate (≥ 99%) used in neutral detergent fibre (NDF) and acid
detergent fibre (ADF) analyses were purchased from Sigma Corporation and
Sigma-Aldrich, respectively. Polyester bags (10 × 24 cm, pore size of 48
µm) for <i>in situ</i> digestion were purchased from Allied Filter Fabrics
Corp, Australia.
</p>
</div>
</div>

<div id="outline-container-orgdde9662" class="outline-3">
<h3 id="orgdde9662"><span class="section-number-3">4.2</span> <a href="#orgba9300a"><b>2.2 Fodder compositional analysis</b></a> <label for="3" class="margin-toggle sidenote-number"></label><input type="checkbox" id="3" class="margin-toggle"/><span class="sidenote">
Can this section be shortened by referring to previous papers?
</span></h3>
<div class="outline-text-3" id="text-4-2">
<p>
Fodder samples were dried at 45 °C for 48 h and milled using a Retsch
SM100 hammer mill (Retsch GmBH, Germany) fitted with a 2 mm sieve. Water
extractives were removed from the milled fodder using a Dionex ASE 350
and the biomass composition of extractive-free fodder was determined
using the NREL method (Sluiter et al. 2008). Briefly, extractive-free
fodder samples were transferred to pressure tubes (Ace Glass
Incorporated, USA), an aliquot (3 mL) of H<sub>2</sub>SO<sub>4</sub> (72% (w/v)) was
added to each tube, and the mixtures were incubated in a water bath at
30 °C for 60 min with mixing via a stirring rod every 10 min. The tubes
were removed from the water bath and 4 mL of ultrapure water was added
to each tube. Sugar recovery standard was prepared by dissolving 0.2 g
D-glucose (≥99.5%), 0.07 g D-xylose (≥99%), 0.02 g D-galactose (≥99%),
and 0.02 g D-arabinose (≥98%) in 100 mL of water. The pressure tubes and
sugar recovery standard were autoclaved at 115 °C for 60 min. The tubes
were then cooled to room temperature and the hydrolysate was filtered
using pre-weighed filtering crucibles. The filtrate was collected for
the quantification of mono- and di-saccharides. The absorbance at 240 nm
of a sub-sample of the filtrate was measured using a UV-Visible
spectrophotometer (Cary 60 UV-Vis, Agilent Technologies) and used to
quantify acid-soluble lignin (ASL) content. The filtering crucibles
containing unhydrolysed solids were dried at 105 °C for 24 h, heated in
muffle furnace (C.T Moloney Pty. Ltd., Sydney) at 575 ± 25 °C for 4 h,
and the mass of the residue was measured to quantify acid-insoluble
lignin (AIL) content. Monomeric sugars were quantified using a HPLC
system (Waters) equipped with a refractive index detector (Waters 410,
US) and a SP810 carbohydrate column (300 mm × 8.0 mm, Shodex, Japan).
The column temperature was 85 °C and water was used as the mobile phase
at a flow rate of 0.5 mL min<sup>-1</sup>. The concentrations of monomeric
sugars in the acid hydrolysate were used to calculate the amounts of
cellulose and hemicellulose in the original biomass sample.
</p>

<p>
NDF and ADF analyses were carried out using an ANKOM 200 Fibre Analyser
according to the manufacturer's methodology. Briefly, fodder samples
(0.45 &#x2014; 0.5 g, dried at 45 °C and milled through a 2 mm screen) were
transferred to ANKOM bags and then placed in the fibre analyser chamber.
NDF solution (100 mL per bag), 8.0 mL of α-amylase enzyme solution, and
sodium sulphate (0.5 g per 50 mL of NDF solution) were added to the
chamber. Samples were incubated and agitated in NDF solution at 100 °C
for 75 min followed by three rinse steps with distilled water at 70 &#x2013;
90 °C for 5 min. Samples were air-dried overnight, dried in an oven at
105 °C for a maximum of 4 h, and then weighed. Subsequently, samples
were added to the vessel and incubated in ADF solution at 100 °C for 60
min and then rinsed with 70 &#x2013; 90 ℃ deionised water in the vessel with
agitation until the samples were neutralised. Bags of samples were
air-dried, incubated in an oven at 105 °C for 2-4 h, and then weighed.
</p>

<p>
Elemental analysis was used to quantify the nitrogen content in the
fodder samples. Fodder samples were dried at 45 °C until their weight
was constant and milled using a Retsch SM100 hammer mill fitted with a 2
mm sieve. Samples were ground using tube mill (IKA, 4180001) and then
analysed for C, N, and S content method using the LECO TruMac Carbon
Nitrogen Sulphur elemental macro analyser (TruMac® CNS, LECO
Corporation, USA). The nitrogen content (g 100 g<sup>-1</sup>) in fodder samples
was used to determine their crude protein content (nitrogen content ×
6.25 = crude protein).
</p>
</div>
</div>

<div id="outline-container-org7f2f740" class="outline-3">
<h3 id="org7f2f740"><span class="section-number-3">4.3</span> <b>2.3 <i>In situ</i> incubation of fodder in the bovine rumen</b></h3>
<div class="outline-text-3" id="text-4-3">
<p>
The <i>in situ</i> rumen incubation of fodder samples was approved by the
animal ethics committees of the University of Queensland and Queensland
University of Technology. Fodder samples were dried at 45 °C for 48 h,
ground using a Retsch SM100 hammer mill (Retsch GmBH, Germany) fitted
with a 2 mm screen, and passed over a 53 µm screen. Sub-samples of
fodder (~5 g) retained on the screen were transferred into polyester
bags with a pore size of 48 µm (Allied Filter Fabrics Corp, Australia)
and placed in the rumen of steers at the Gatton campus of the University
of Queensland. Two fistulated steers were used to characterise bacterial
colonisation of fodder. The <i>Bos indicus</i> steers were fed on green couch
grass and sorghum hay as supplement <i>ad libitum</i>. Each steer hosted 60
bags of fodder ((2 substrates × 10 time points) × 6 replicates). The
nylon bags were collected from the rumen after 4, 8, 12, 18, 24, 32, 40,
48, 60, <a href="#orgd62f450">and 72 h of incubation</a>, washed thoroughly with water until the water
was clear, and sub-samples of the residual material in each bag were
collected and stored at -80 °C for DNA extraction and full-length 16S
PacBio DNA sequencing. The bags and their remaining contents were then
dried at 65 °C and analysed for dry matter (DM), NDF, and ADF content.
Rumen fluid samples (~15 mL) were collected from both steers at each
time point, were immediately snap-frozen with liquid nitrogen, and then
stored at -20 °C prior to DNA extraction.
</p>
</div>
</div>

<div id="outline-container-org0c9d980" class="outline-3">
<h3 id="org0c9d980"><span class="section-number-3">4.4</span> <b>2.4 Fodder analysis post-digestion</b></h3>
<div class="outline-text-3" id="text-4-4">
<p>
DM digestibility of fodders was analysed gravimetrically. Four bags from
each timepoint during <i>in situ</i> rumen incubation were dried in a vacuum
oven at 60 °C until a constant weight was reached. DM digestibility was
calculated from the difference in dry mass of the original and fermented
samples divided by original sample mass. Fermented samples (0.45 &#x2014; 0.5
g) were transferred and sealed in ANKOM bags to undertake sequential NDF
and ADF analyses using an ANKOM 200 Fibre Analyser (ANKOM Technology
Corp, US) as described above.
</p>
</div>
</div>

<div id="outline-container-org1ed82a6" class="outline-3">
<h3 id="org1ed82a6"><span class="section-number-3">4.5</span> <b>2.5 DNA extraction</b></h3>
<div class="outline-text-3" id="text-4-5">
<p>
Total genomic DNA from fodder, fermented fodder, and rumen fluid samples
was extracted using the <a href="#org30a74d3">Soil DNA Isolation Mini Kit (FAVORGEN)</a> <label for="4" class="margin-toggle sidenote-number"></label><input type="checkbox" id="4" class="margin-toggle"/><span class="sidenote">
Is this kit suitable for rumen microbiome analysis - references?
</span> as per
the manufacturer's instructions and cell disruption during the DNA
extraction procedure was undertaken using a Qiagen TissueLyser II
(Retsch, 30 Hz). The quantity and quality of total genomic DNA were
measured using a NanoDrop Microvolume Spectrophotometer (ND-2000, Thermo
Fisher Scientific). High-quality DNA samples with the absorbance 260
nm/280 nm ratios of 1.8 &#x2013; 2.0 and 260 nm/230 nm ratios of 2.0 &#x2013; 2.2
were used for PCR reactions.
</p>
</div>
</div>

<div id="outline-container-orga8247e9" class="outline-3">
<h3 id="orga8247e9"><span class="section-number-3">4.6</span> <b>2.6 Full-length 16S rRNA amplification and PacBio sequencing</b></h3>
<div class="outline-text-3" id="text-4-6">
<p>
PacBio 16S rRNA gene sequencing was undertaken according to Procedure &amp;
Checklist - Full-Length 16S Amplification, SMRTbell® Library Preparation
and Sequencing (Version 1). Amplicons of full-length 16S ribosomal RNA
genes were generated for each sample using universal primers 27F
(5'-AGRGTTYGATYMTGGCTCAG -3') and 1492R (5'- RGYTACCTTGTTACGACTT -3')
containing a PacBio universal sequence overhang. PCR amplification was
performed in an Eppendorf Thermal Cycler (Germany) using 2.5 ng of
template DNA (2.5 ng µL<sup>-1</sup>), 0.75 µL of each primer (10 µM), and 12.5
µL of KAPA HiFi Hot Start DNA Polymerase (Sigma-Aldrich, Sydney,
Australia). The PCR amplification conditions were as follows: 20 cycles
of 95 °C for 30 sec, 57 °C for 30 sec, and 72 °C for 60 sec and held at
4 °C. PacBio barcoded primers were added to the primary PCR amplicons in
a secondary PCR using 1 ng of template DNA (1 ng µL<sup>-1</sup>). The thermal
cycling procedure for the secondary PCR was the same as those of the
primary PCR with the following alterations: 15 cycles and extension
times of 120 sec were used to avoid chimera formation. The secondary PCR
products were resolved by agarose gel electrophoresis (1% (w/v) agarose,
45 min at 90 kV). PCR amplicons were then purified using AMPure PB beads
according to the manufacturer's instructions. The concentrations of
purified PCR products were measured using a Qubit 3.0 Fluorometer
(Thermo Fisher Scientific). Purified, barcoded PCR amplicons were then
pooled in equimolar concentrations and the library was constructed using
SMRTbell™ Template Prep Kit v1.0-SPv3 according to the manufacturer's
protocol. The size and quality of the library were evaluated using a
Bioanalyser (Agilent 2100) and library sequencing was performed on a
PacBio Sequel platform using 1M SMRT Cells. Sequencing Primer v3,
Sequel™ Binding Kit 3.0, and Sequel DNA Internal Control complex 3.0
were used for the binding reaction.
</p>
</div>
</div>

<div id="outline-container-orgc7bd8f4" class="outline-3">
<h3 id="orgc7bd8f4"><span class="section-number-3">4.7</span> <b>2.7 Bioinformatics and statistical analyses</b></h3>
<div class="outline-text-3" id="text-4-7">
<p>
SMRT Link software (version 9.0.0) was used to process raw PacBio
sequencing data, generate the Circular Consensus Sequence (CCS) reads,
and demultiplex samples. The CCS sequences were then assigned to
corresponding samples based on their unique PacBio barcodes using the
lima tool (version 1.11.0). The output files from the lima tool were
then imported to the Ampliseq pipeline for analysis that includes the
following workflow (Straub et al. 2020): (i) examination of quality
control of sequences using the FastQC tool (Andrews 2010), (ii) trimming
adapter sequences from sequencing reads using the Cutadapt tool (Martin
2011), (iii) importation of the data into QIIME2 (Bolyen et al. 2019),
(iv) generation of amplicon sequencing variants (ASV) using DADA2
(Callahan et al. 2016), and (v) taxonomic classification based on SILVA
v132 database (Quast et al. 2012). Downstream analysis was conducted in
R version 4.0.5 (2021-03-31) (Core 2013) and the ampvis2 package was
used to visualise the sequencing data (Andersen et al. 2018).
</p>

<p>
The α-diversity metrics were conducted using Shannon's diversity index
and Observed ASVs (Shannon and Weaver 1949, DeSantis et al. 2006). A
Kruskal-Wallis and Wilcoxon rank sum tests were used to analyse the
differences between treatment groups for diversity indices. The
<i>P</i>-values were adjusted according to the Benjamini-Hochberg method. The
principal coordinate analysis (PCoA) was performed using Bray&#x2013;Curtis
distance metrics to visualise differences in bacterial communities
between timepoints, phases of digestion, and fodder types. The overall
and pairwise significance between different groups were analysed using a
Permutational Multivariate Analysis of Variance (PERMANOVA) in R vegan
package (Dixon 2003).
</p>

<p>
The significant differences in the kinetics of digestion of sugarcane
bagasse and Lucerne hay during incubation in the bovine rumen and
changes in the taxa abundances between timepoints and phases of
digestion were evaluated in R. The normality and homogeneity of
variances were tested using D'agostino-Pearson and Levene's tests,
respectively. The non-parametric Kruskal-Wallis test was used for
independent samples and pairwise comparisons were performed with the
Dunn's post-hoc test and <i>P</i>-values were adjusted using the
Benjamini-Hochberg method. Statistical significance was declared when
/P/≤0.05. These analyses were conducted in R using fBasics (Wuertz et
al. 2020), and FSA (Ogle et al. 2021) packages. The overall significant
differences in the bacterial communities between fodders were evaluated
using the Analysis of Composition of Microbiomes with Bias Correction
(ANCOM-BC) package (Mandal et al. 2015) in R and <i>P</i>-values were
adjusted using the Holm&#x2013;Bonferroni method.
</p>
</div>
</div>
</div>

<div id="outline-container-org07d8dbd" class="outline-2">
<h2 id="org07d8dbd"><span class="section-number-2">5</span> <b>3. Results</b></h2>
<div class="outline-text-2" id="text-5">
</div>
<div id="outline-container-org6cde82e" class="outline-3">
<h3 id="org6cde82e"><span class="section-number-3">5.1</span> <b>3.1 Fodder composition</b></h3>
<div class="outline-text-3" id="text-5-1">
<p>
Bagasse is the fibrous residue remaining after sugarcane (<i>Saccharum
officinarum</i>) stalk billets are processed in a mill to extract sugar
juice. The goal of commercial sugarcane milling is to rupture every cell
in the stalk billet and release the maximum amount of sugar juice. In
contrast, Lucerne (<i>Medicago sativa</i>) hay is harvested, dried, and baled
without significant cell disruption. The difference in cell disruption
during processing is apparent in the composition of the two fodders
(Table 1). The NDF and ADF content in sugarcane bagasse was
significantly higher than those of Lucerne hay (<i>P</i> &lt; 0.01) but
sugarcane bagasse contained significantly less crude protein and ash
than that of Lucerne hay (<i>P</i> &lt; 0.001). Despite the significant
differences in extractives, crude protein, and ash contents, the
cellulose, hemicellulose, and lignin contents in sugarcane bagasse and
Lucerne hay fibres were similar.
</p>
</div>
</div>

<div id="outline-container-org83f2fa0" class="outline-3">
<h3 id="org83f2fa0"><span class="section-number-3">5.2</span> <b>3.2 Degradation of sugarcane bagasse and Lucerne hay in the bovine rumen</b></h3>
<div class="outline-text-3" id="text-5-2">
<p>
Sugarcane bagasse and Lucerne hay were incubated in the bovine rumen and
samples were removed after 4 &#x2013; 72 h for DM, NDF, and ADF analyses
(Figure 1). <a href="#orgb16cebd">The results of these analyses demonstrated that the digestion kinetics of sugarcane bagasse and Lucerne hay were significantly different</a>. <label for="5" class="margin-toggle sidenote-number"></label><input type="checkbox" id="5" class="margin-toggle"/><span class="sidenote">
P-value to back this up?
</span>
DM and fibre (NDF and ADF) digestion in
sugarcane bagasse were similar in bovine rumen fluid because of the
relatively high (89%) fibre content therein. <a href="#org5dfa9ef">Sugarcane bagasse fibre digestion occurred in three phases: an initial, relatively rapid phase from 0 &#x2013; 12 h, an intermediate phase from 12 &#x2013; 40 h, and a relatively slow phase from 40 &#x2013; 72 h (Figure 1, Panel B and C)</a>. <label for="6" class="margin-toggle sidenote-number"></label><input type="checkbox" id="6" class="margin-toggle"/><span class="sidenote">
For sugarcane bagasse, the rate appears constant to me
</span>
It should be noted
that digestion of sugarcane bagasse did not reach a maximum (<i>i.e.</i>,
so-called plateau phase) within 72 h. <a href="#org4e97962">In contrast, the DM and fibre digestion kinetics in Lucerne hay were significantly different because of relatively high (~40%) initial dry matter digestion without fibre digestion and that digestion of Lucerne hay reached a maximum at 48 h</a> <label for="7" class="margin-toggle sidenote-number"></label><input type="checkbox" id="7" class="margin-toggle"/><span class="sidenote">
Maximum for lucerne hay is less than 48 h. More like 32 h
</span>
(Figure 1). Lucerne hay fibre digestion occurred in four phases: a
intitial solubilisation phase (4 h), a relatively rapid digestion phase
from 4 &#x2013; 12 h, a relatively slow digestion phase from 12 &#x2013; 32 h, and a
plateau phase from 32 &#x2013; 72 h (Figure 1, Panel B and C).
</p>
</div>
</div>

<div id="outline-container-org44563d4" class="outline-3">
<h3 id="org44563d4"><span class="section-number-3">5.3</span> <a href="#org8bfcf53"><b>3.2 Temporal changes in the bacterial community during bovine rumen degradation of sugarcane bagasse and Lucerne hay</b></a> <label for="8" class="margin-toggle sidenote-number"></label><input type="checkbox" id="8" class="margin-toggle"/><span class="sidenote">
This should be section 3.3
</span></h3>
<div class="outline-text-3" id="text-5-3">
<p>
Temporal changes in the bacterial community during sugarcane bagasse and
Lucerne hay digestion were measured using full-length 16S rRNA gene
sequencing. DNA was extracted from sugarcane bagasse, Lucerne hay, and
rumen fluid at each timepoint during digestion. 16S rRNA gene sequencing
of all samples generated 2,122,602 CCS reads and 85,403 ASVs, with an
average of 11,012 ± 2,054 and 3,425 ± 655 ASVs per sample. The total
number of post-filtering CCS sequences for all samples was 1,596,720
with an average of 88% of 16S rRNA gene sequences classified to a
specific bacterial genus and an average of 54% classified to an
individual species.
</p>

<p>
The absolute (alpha) diversity of the bacterial communities in the
fodder samples from <i>in situ</i> digestion in bovine rumen fluid was
expressed numerically as Shannon's and <a href="#orgc7b8142">Observed ASVs</a> <label for="9" class="margin-toggle sidenote-number"></label><input type="checkbox" id="9" class="margin-toggle"/><span class="sidenote">
Should this be Observed OTUs?
</span> indices (Figure 2).
The α-diversity of the bacterial communities on the surface of sugarcane
bagasse and Lucerne hay particles increased significantly after
incubation in the rumen relative to the epiphytic (0 h) bacterial
community (<a href="#org1e72437">data not shown</a>) (<i>P</i> &lt; 0.04). The average diversity of the
bacterial communities on sugarcane bagasse was significantly lower than
Lucerne hay during <i>in situ</i> incubation in the rumen, (<i>P</i> &lt; 0.001)
(Figure 2A).
</p>

<p>
The kinetics of α-diversity of bacterial communities on sugarcane
bagasse and Lucerne hay varied significantly during incubation in the
bovine rumen (<i>P</i> &lt; 0.0001) (Figure 2B). The α-diversity of bacterial
communities adherent to sugarcane bagasse did not change from 4 &#x2013; 8 h
of incubation (<i>P</i> = 0.30). However, it significantly decreased between
8 h and 12 h (<i>P</i> &lt; 0.02), increased between 18 h and 24 h (<i>P</i> &lt; 0.03),
and then decreased between 24 h and 40 h (<i>P</i> &lt; 0.01). The α-diversity
of bacterial communities on sugarcane bagasse changed variable after 40
h of incubation (<i>P</i> &lt; 0.01). The α-diversity of bacterial communities
adherent to Lucerne hay samples increased from 4 &#x2013; 18 h (<i>P</i> &lt; 0.02)
and then did not change significantly between 18 - 72 h (<i>P</i> &gt; 0.1)
except between 18 h to 48 h and 48 h to 60 h (<i>P</i> &lt; 0.01). <a href="#org402d330">Finally, the alpha diversity of the planktonic bacteria did not differ significantly during the 72 h of the experiment (<i>P</i> = 0.1)</a> <label for="10" class="margin-toggle sidenote-number"></label><input type="checkbox" id="10" class="margin-toggle"/><span class="sidenote">
Data not shown? Why? This might be a good comparison
</span>,
indicating that
differences in the absolute diversity of the microbial communities on
low- and high-quality fodder were a function of their inherent physical,
chemical, and nutritional characteristics. Alpha diversity data showed
no significant differences in bacterial diversity when comparing
sugarcane bagasse and Lucerne hay at 4 h and 8 h of incubation (<i>P</i> &gt;
0.08) (Figure 2B). However, sugarcane bagasse had significantly lower
alpha diversity between 12 h and 40 h and after 60 h of incubation than
Lucerne hay (P &lt; 0.02).
</p>

<p>
A comparison of α-diversity of bacterial communities adherent to
sugarcane bagasse between phases of digestion revealed that the
diversity of bacterial communities on sugarcane bagasse decreased
significantly between rapid digestion and intermediate phases (<i>P</i> =
0.05) and then did not change between intermediate and slow phases of
digestion (<i>P</i> = 0.9) (Figure 2C). The α-diversity of bacterial
communities adherent to Lucerne hay increased significantly between
solubilisation and rapid digestion phases (<i>P</i> = 0.0005), increased
significantly between rapid and slow digestion phases (<i>P</i> = 0.02), and
then did not change between slow digestion and plateau phases of
digestion (<i>P</i> &gt; 0.23). A comparison of the alpha diversity of bacterial
communities colonising sugarcane bagasse and Lucerne hay revealed that
no significant difference between the diversity of bacterial communities
adherent to sugarcane bagasse and lucerne during rapid fibre digestion
phase was observed (<i>P</i> = 0.1); however, Sugarcane bagasse had
significantly lower diversity than Lucerne hay in the intermediate and
slow digestion phases compared to slow digestion and plateau phases in
lucerne hay (<i>P</i> &lt; 0.001). The correlation between the α-diversity of
bacterial community adherent to sugarcane bagasse and Lucerne hay and
NDF digestibility revealed that increasing NDF digestibility of
sugarcane bagasse was correlated to a decrease in bacterial diversity,
while the bacterial complexity associated with Lucerne hay increased
with increasing NDF digestibility (Figure S1).
</p>

<p>
<a href="#org05ca604">** Beta diversity analysis</a> <label for="11" class="margin-toggle sidenote-number"></label><input type="checkbox" id="11" class="margin-toggle"/><span class="sidenote">
New subheading? Beta diversity analysis
</span>
</p>

<p>
The relative differences in the composition of the microbial communities
(β diversity) in the fodder samples between timepoints during digestion
were evaluated. A PCoA plot of Bray-Curtis dissimilarity was used to
visualise the relative difference in the bacterial communities on
sugarcane bagasse and Lucerne hay (Figure 3A and 3B). The epiphytic
sessile bacterial communities on both sugarcane bagasse and Lucerne hay
were significantly different from the sessile bacterial communities
associated with the fodders after rumen incubation (<i>P</i> &lt; 0.02) (data
not shown in the figure). The composition of the bacterial communities
on sugarcane bagasse and Lucerne hay changed significantly over time
(<i>P</i> = 0.001) (Figure 3A and 3B) and sugarcane bagasse had a distinct
bacterial community from those on the surface of Lucerne hay in each
incubation timepoint (<i>P</i> &lt; 0.04). Overall, the bacterial communities colonising sugarcane bagasse were significantly different from the colonising microbiota of Lucerne hay (<i>P</i> = 0.001) <a href="#orgf94aaff">(Figure S2)</a> <label for="12" class="margin-toggle sidenote-number"></label><input type="checkbox" id="12" class="margin-toggle"/><span class="sidenote">
I'm not sure how to interprete this figure
</span>. The
comparison of beta diversity of bacterial communities colonising
sugarcane bagasse in different phases of digestion revealed that
bacterial communities were distinct in each phase of digestion (<i>P</i> =
0.001) (Figure 3C). Further, the beta diversity results showed four
significantly different clusters of bacterial groups on the surface of
Lucerne hay based on phases of digestion (<i>P</i> = 0.001) (Figure 3D). A
comparison of beta diversity between the bacterial communities
colonising sugarcane bagasse and Lucerne hay in each phase of digestion
indicated that the bacterial groups on sugarcane bagasse differed
significantly from those attached to Lucerne hay in each phase of
digestion (<i>P</i> &lt; 0.001). These results also confirm that differences in
the composition of bacterial communities colonising sugarcane bagasse
and Lucerne hay were a function of their inherent physical, chemical,
and nutritional characteristics.
</p>
</div>

<div id="outline-container-org5bd827d" class="outline-4">
<h4 id="org5bd827d"><span class="section-number-4">5.3.1</span> Epiphytic bacterial communities on sugarcane bagasse and Lucerne hay</h4>
<div class="outline-text-4" id="text-5-3-1">
<p>
The bacterial community on the surface of sugarcane bagasse and Lucerne
hay before incubation in the rumen was evaluated (Figure S3). The main
phyla of epiphytic bacteria on sugarcane bagasse were Firmicutes and
Proteobacteria, and the most abundant bacterial genera were
<i>Tumebacillus</i>, <i>Cohnella</i>, <i>Bacillus</i>, and <i>Massilia</i>. In contrast, the
main phyla of epiphytic bacteria on Lucerne hay were Cyanobacteria and
Proteobacteria/,/ and the most abundant bacterial genera (excluding the
bacterial genus related to Cyanobacteria taxa) were <i>Massilia</i>,
<i>Falsirhodobacter</i>, <i>Stenotrophomonas</i>, <i>Pseudomonas</i>, <i>Paracoccus</i>,
<i>Sphingomonas</i>, and <i>Pantoea</i>. The Cyanobacteria taxa was related to the
sequencing of plant chloroplast which is the main issue with sequencing
of 16S rRNA gene of fodder samples. After 4 h of incubation in the
bovine rumen, the relative abundance of these bacterial genera decreased
significantly on both sugarcane bagasse and Lucerne hay (<i>P</i> &lt; 0.05),
thereby providing direct evidence for the replacement of the epiphytic
bacterial community with those from the planktonic rumen bacterial
community.
</p>
</div>
</div>

<div id="outline-container-org7dfb0d5" class="outline-4">
<h4 id="org7dfb0d5"><span class="section-number-4">5.3.2</span> Dynamics of changes in the bacterial community adherent to sugarcane bagasse and Lucerne hay during bovine rumen incubation</h4>
<div class="outline-text-4" id="text-5-3-2">
<p>
A total of 25 bacterial phyla were identified in samples of fodder fermented in the bovine rumen. Firmicutes and Bacteroidetes were the main bacterial phyla that colonised the surface of sugarcane bagasse and Lucerne hay during <i>in situ</i> incubation and the relative abundance of these bacterial phyla changed significantly during incubation (<i>P</i> lt 0.001) (Figure 4). The relative abundance of Tenericutes, Fibrobacteres, and Spirochaetes on the surface of sugarcane bagasse and Lucerne hay also changed significantly during incubation (<i>P</i> &lt; 0.001);
as a result, the genera of bacteria on the surface of sugarcane bagasse that changed
significantly during incubation in the rumen were <i>Prevotella 1</i>,
<i>Rikenellaceae RC9</i> gut group, <i>Lachnospiraceae XPB1014</i> group,
<i>Ruminococcus 1</i>, <i>Christensenellaceae R-7 group</i>, <i>Fibrobacter</i>,
<i>Treponema 2</i>, uncultured <i>Lachnospiraceae</i>, uncultured rumen bacterium
in the <i>p-251-o5</i> family, and <i>Saccharofermentans</i> (<i>P</i> &lt; 0.05). The
genera of bacteria on the surface of Lucerne hay that changed
significantly in abundance during incubation in the rumen were
<i>Butyrivibrio 2</i>, the <i>Christensenellaceae R-7 group</i>, the
<i>Rikenellaceae RC9</i> gut group, <i>Ruminococcus 1</i>, <i>Treponema 2</i>,
<i>Saccharofermentans</i>, the <i>Lachnospiraceae XPB1014 group</i> (<i>P</i> &lt; 0.05).
</p>

<p>
<a href="#org3a81e63">A comparison of the bacterial communities colonising sugarcane bagasse during incubation in the rumen revealed that the relative abundance of Firmicutes and Bacteroidetes remained relatively high during incubation and their relative abundance did not change consistently during the incubation. The relative abundance of Fibrobacteres increased significantly between 18 h and 32 h compared to 4 h incubation (<i>P</i> &lt; 0.02) and the relative abundance of Spirochaetes increased significantly between 24 h and 72 h incubation compared to 4 h (<i>P</i> &lt; 0.03). The relative abundance of Firmicutes decreased significantly (<i>P</i> = 0.001) on Lucerne hay during <i>in situ</i> incubation, while Bacteroidetes increased in abundance (<i>P</i> = 0.001). The relative abundance of Spirochaetes increased significantly on Lucerne hay between 12 h and 48 h compared with 4 h of incubation (<i>P</i> &lt; 0.032) and a higher abundance of Tenericutes was observed between 8 h and 24 h compared to 4 h of incubation (<i>P</i> &lt; 0.04).</a> <label for="13" class="margin-toggle sidenote-number"></label><input type="checkbox" id="13" class="margin-toggle"/><span class="sidenote">
I don't think this paragraph is necessary as the focus should be on genus/species-level analysis
</span>
</p>

<p>
The composition of bacterial genera colonising sugarcane bagasse changed during incubation in the rumen <a href="#orgd9c1d98">(Figure 5)</a> <label for="14" class="margin-toggle sidenote-number"></label><input type="checkbox" id="14" class="margin-toggle"/><span class="sidenote">
Perhaps just include the genera that significantly changed in this figure?
</span>. <i>Christensenellaceae R-7
group</i>, <i>Prevotella 1</i>, and <i>Ruminococcus 1</i> were the most abundant
bacterial genera on the surface of sugarcane bagasse after 4 h of
incubation in the rumen. The relative abundance of <i>Prevotella 1</i>
decreased significantly during the incubation (<i>P</i> = 0.001), while the
relative abundance of <i>Christensenellaceae R-7 group</i> remained
relatively unchanged <a href="#org0390f73">(<i>P</i> = 0.06)</a> <label for="15" class="margin-toggle sidenote-number"></label><input type="checkbox" id="15" class="margin-toggle"/><span class="sidenote">
Don't need to show this P value
</span>. The relative abundance of
<i>Ruminococcus 1</i> remained relatively high between 4 h and 32 h of
incubation; however, its relative abundance decreased significantly
after 40 h compared with 4 h of incubation (<i>P</i> &lt; 0.002). The relative abundance of <i>Lachnospiraceae XPB1014 group</i> remained relatively unchanged between 8 h and 32 h <a href="#orgf0383dd">(<i>P</i> &gt; 0.0)</a> <label for="16" class="margin-toggle sidenote-number"></label><input type="checkbox" id="16" class="margin-toggle"/><span class="sidenote">
Greater than 0.0 what?
</span> but decreased significantly after 40 h compared to 8 h of incubation (<i>P</i> &lt; 0.03). <i>Rikenellaceae RC9 gut group</i> and uncultured rumen bacterium in the <i>p-251-o5</i> <i>gut group</i> family were enriched after 12 h compared with 4 h <a href="#orgee97a60">(<i>P</i> &lt;)</a> <label for="17" class="margin-toggle sidenote-number"></label><input type="checkbox" id="17" class="margin-toggle"/><span class="sidenote">
Less than what?
</span>.
<i>Treponema 2</i> increased significantly in abundance after 24 h (<i>P</i> &lt;
0.02).
</p>

<p>
<a href="#org0e445c6">As was the case with rumen-incubated sugarcane bagasse samples, the composition of bacterial genera colonising Lucerne hay changed significantly during rumen digestion (Figure 5). Bacteria from the genera <i>Christensenellaceae R-7 group</i>, <i>Butyrivibrio 2, Prevotella 1</i>, and <i>Lachnospiraceae XPB1014 group</i> were dominant on the surface of Lucerne hay after 4 h of digestion. The relative abundance of <i>Prevotella 1</i> remained unchanged between 4 &#x2013; 48 h (<i>P</i> &gt; 0.1) and decreased significantly after 48 h incubation compared to 8 h of incubation (<i>P</i> &lt; 0.009). The relative abundance of <i>Christensenellaceae R-7 group</i> remained relatively high during the incubation, though its relative abundance decreased at 8 h compared to 4 h incubation (<i>P</i> = 0.045), and then remained unchanged during the incubation (<i>P</i> &gt; 0.2). <i>Butyrivibrio 2</i> remained relatively abundant throughout digestion, although it decreased significantly after 48 h compared with 4 h (<i>P</i> &lt; 0.04). <i>Rikenellaceae RC9 gut group</i> was enriched after 24 h incubation in the rumen (<i>P</i> &lt; 0.03) and <i>Saccharofermentans</i> increased significantly in abundance after 18 h compared to 4 h (<i>P</i> &lt; 0.03). The relative abundance of <i>Treponema 2</i> increased significantly between 12 and 48 h compared to 4 h (<i>P</i> &lt; 0.029) and decreased significantly after 60 h compared to 32 h (<i>P</i> &lt; 0.04). The relative abundance of <i>Ruminococcus 1</i> increased between 4 h and 24 h but decreased significantly after 48 h compared to 24 h (P &lt; 0.02). The relative abundance of <i>Lachnospiraceae XPB1014 group</i> remained high between 4 h and 18 h but decreased significantly after 48 h compared to 4 h (<i>P</i> &lt; 0.04). <i>Lachnospiraceae NK4A136 group</i> were prevalent at 4 and 8 h of incubation but their relative abundance decreased significantly after 12 h (<i>P</i> &lt; 0.05)</a> <label for="18" class="margin-toggle sidenote-number"></label><input type="checkbox" id="18" class="margin-toggle"/><span class="sidenote">
Is it possible to just contrast these findings with the bagasse findings and highlight differences? It all sounds very repetitive the way it is currently written
</span>.
</p>
</div>
</div>

<div id="outline-container-org076bd47" class="outline-4">
<h4 id="org076bd47"><span class="section-number-4">5.3.3</span> <a href="#orgd1be9b5">Comparison of bacterial communities adherent to sugarcane bagasse and Lucerne hay</a> <label for="19" class="margin-toggle sidenote-number"></label><input type="checkbox" id="19" class="margin-toggle"/><span class="sidenote">
How is this section different to the previous one? Can the important comparisons be incorporated into one section? Primarily focussing on species/genus-level analysis
</span></h4>
<div class="outline-text-4" id="text-5-3-3">
<p>
The overall bacterial composition on the surface of sugarcane bagasse
and Lucerne hay during incubation in the rumen was characterised (Figure
S4). The bacterial phyla that were affected by fodder type include
Fibrobacteres, Tenericutes, and Proteobacteria (<i>P</i> &lt; 0.05). On a genus
level, <i>Fibrobacter</i>, <i>Saccharofermentans</i>, <i>Butyrivibrio 2</i>,
<i>Lachnospiraceae AC2044 group</i>, and <i>Anaeroplasma</i> were significantly
different in abundance between sugarcane bagasse and Lucerne hay (P &lt;
0.05).
</p>

<p>
A significantly higher percentage of Firmicutes and a lower abundance of
Bacteroidetes was observed in Lucerne hay samples at 4 h of incubation
compared with sugarcane bagasse (<i>P</i> &lt; 0.04). However, no significant
differences in the abundance of Firmicutes and Bacteroidetes were
observed between Lucerne hay and sugarcane bagasse after 8 h incubation
(<i>P</i> &gt; 0.05). <a href="#org8a09982">The proportion of Fibrobacteres was significantly higher in sugarcane bagasse at 24 h of incubation, whereas Lucerne hay had a higher abundance of Tenericutes at 8, 12, and 18 h of incubation</a> (<i>P</i> &lt; 0.05).
</p>

<p>
<a href="#orgc5ccd89">A higher abundance of <i>Prevotella 1</i>, <i>Ruminococcus 1</i>, and <i>Rikenellaceae RC9 gut group</i> were observed in sugarcane bagasse samples (<i>P</i> lt 0.05) at 4 h of incubation, while <i>Butyrivibrio 2</i>, <i>Christensenellaceae R-7 group</i>, and <i>Lachnospiraceae XPB1014 group</i> were more abundant in Lucerne hay samples (<i>P</i> lt 0.03). Sugarcane bagasse samples presented a significantly higher percentage of <i>Saccharofermentans</i> after 8 h of incubation compared with Lucerne hay (<i>P</i> lt 0.05), while Lucerne hay samples had a significantly higher abundance of <i>Butyrivibrio 2</i> after 8 h (<i>P</i> lt 0.1). <i>Anaeroplasma</i> was significantly more abundant on the surface of Lucerne hay than sugarcane bagasse between 8 h and 18 h of incubation (<i>P</i> lt 0.04). <i>Fibrobacter</i> was significantly more abundant on the surface of sugarcane bagasse compared with Lucerne hay at 24 h of incubation. Sugarcane bagasse had a significantly higher percentage of <i>Treponema 2</i> at 48 h, while <i>Treponema 2</i> was significantly more abundant in Lucerne hay samples at 18 h of incubation.</a> <label for="20" class="margin-toggle sidenote-number"></label><input type="checkbox" id="20" class="margin-toggle"/><span class="sidenote">
Is there a figure highlighting these differences?
</span>
</p>
</div>
</div>

<div id="outline-container-org3e6a57e" class="outline-4">
<h4 id="org3e6a57e"><span class="section-number-4">5.3.4</span> <a href="#org2c35723">Correlation between degradation of sugarcane bagasse and Lucerne hay and bacterial colonisation during <i>in situ</i> incubation</a> <label for="21" class="margin-toggle sidenote-number"></label><input type="checkbox" id="21" class="margin-toggle"/><span class="sidenote">
Again, this is all quite repetitive. I wonder if this and the previous two sections can be condensed into one, highlighting only the major differences? This will help keep the reader engaged
</span></h4>
<div class="outline-text-4" id="text-5-3-4">
<p>
The bacterial genera in the communities on the surface of sugarcane
bagasse and Lucerne hay changed significantly in abundance between the
phases of incubation (<i>P</i> &lt; 0.05) (Figure 6). The bacterial genera that
were dominant on sugarcane bagasse in the rapid digestion phase include
<i>Christensenellaceae R-7 group</i>, <i>Prevotella 1</i>, <i>Rikenellaceae RC9 gut
group</i>, <i>Lachnospiraceae XPB1014 group</i>, and <i>Ruminococcus 1</i>. The
relative abundance of <i>Rikenellaceae RC9</i> gut group,
<i>Saccharofermentans</i>, <i>Treponema 2</i>, and uncultured rumen bacterium
within the <i>p-251-o5</i> family increased in the intermediate and slow
digestion phases (<i>P</i> &lt; 0.041), while <i>Prevotella 1</i> and <i>Ruminococcus
1</i> decreased in abundance (<i>P</i> &lt; 0.001). Interestingly, <i>Fibrobacter</i>
increased significantly in abundance between rapid digestion and
intermediated digestion phases (<i>P</i> = 0.0001) but decreased
significantly in the slow digestion phase relative to intermediate
digestion phase (<i>P</i> = 0.005).
</p>

<p>
The bacterial genera on Lucerne hay that dominated the solubilisation
phase were <i>Christensenellaceae R-7 group</i>, <i>Butyrivibrio 2</i>,
<i>Lachnospiraceae NK3A20 group</i>, <i>Lachnospiraceae XPB1014 group</i>, and
<i>Prevotella 1</i>. The relative abundance of <i>Treponema 2</i> and
<i>Anaeroplasma</i> increased significantly in the rapid and slow digestion
phases compared to the solubilisation phase (<i>P</i> &lt; 0.001), but decreased
significantly in the plateau phase relative to slow digestion phase (<i>P</i>
&lt; 0.001). The relative abundance of <i>Lachnospiraceae XPB1014 group</i> and
<i>Lachnospiraceae NK3A20 group</i> decreased significantly in the slow
digestion and plateau phases (<i>P</i> &lt; 0.028), while <i>Saccharofermentans</i>,
<i>Rikenellaceae RC9</i> gut group, and <i>Fibrobacter</i> enriched (<i>P</i> &lt; 0.01).
The relative abundance of <i>Ruminococcus 1</i>, <i>Treponema 2</i>, <i>Butyrivibrio
2</i>, and <i>Prevotella 1</i> decreased significantly in abundance in the
plateau phase compared to slow digestion phase (<i>P</i> &lt; 0.001).
</p>
</div>
</div>

<div id="outline-container-org306b666" class="outline-4">
<h4 id="org306b666"><span class="section-number-4">5.3.5</span> <a href="#org4836c7c">Dynamics of changes in the planktonic bacterial communities</a> <label for="22" class="margin-toggle sidenote-number"></label><input type="checkbox" id="22" class="margin-toggle"/><span class="sidenote">
This section be incorporated into the "Epiphytic bacterial communities on sugarcane bagasse and Lucerne hay" section if you change the title of that section
</span></h4>
<div class="outline-text-4" id="text-5-3-5">
<p>
The planktonic bacterial communities at each incubation time of
sugarcane bagasse and Lucerne hay in the bovine rumen were characterised
and compared to explore if any changes occurred in the composition of
planktonic bacterial community during the incubation of sugarcane
bagasse and Lucerne hay had any impact on the fodder-associated
bacterial community. (Figure 7). The most abundant planktonic bacterial
phyla in the rumen fluid were Firmicutes and Bacteroidetes and their
relative abundance did not change significantly between sampling time
points (Figure S5). The dominant bacterial genera in the rumen fluid
samples include <i>Prevotella 1</i>, <i>Rikenellaceae RC9</i> gut group,
<i>Lachnospiraceae XPB1014 group</i>, <i>Christensenellaceae R-7 group</i>,
uncultured bacterium in the <i>Lachnospiraceae</i> family, uncultured rumen
bacterium in the <i>Bacteroidales BS11 gut group</i> family, <i>Butyrivibrio
2</i>, <i>Lachnospiraceae NK3A20 group</i>, <i>Ruminococcaceae NK4A214 group</i> and
<i>Saccharofermentans</i>. The dominant bacterial genera in the rumen fluid
did not change in abundance between sampling timepoints (<i>P</i> &gt; 0.2).
</p>
</div>
</div>
</div>
</div>

<div id="outline-container-orgaab9a21" class="outline-2">
<h2 id="orgaab9a21"><span class="section-number-2">6</span> <b>4. Discussion</b></h2>
<div class="outline-text-2" id="text-6">
<p>
<a href="#org3745475">Understanding the composition of rumen microbial communities that colonise and proliferate on the surface of fodder particles is essential for understanding fodder digestion. It can ultimately lead to the development of novel methods to increase nutrient use efficiency in ruminants and improve animal productivity (Du et al. 2019). However, only limited studies have investigated the microbial community associated with fodder particles along the kinetics of fodder digestion in the bovine rumen. A comparison of microbial communities that colonise and degrade low- and high-quality fodders provides an opportunity to determine (i) diversity of bacterial communities that colonise low- and high-quality fodders in the bovine rumen, (ii) if the fibre-degrading microbial community in the rumen varies with the source of the fibre, (iii) if the microbial community degrading fibres in the rumen at different phases of fibre digestion vary with the source of fibre, and (iv) whether microbial communities that persist on fibres after fibre digestion has reached a maximum vary with the source of fibre.</a> <label for="23" class="margin-toggle sidenote-number"></label><input type="checkbox" id="23" class="margin-toggle"/><span class="sidenote">
This sounds like it should go in the Introduction
</span>
</p>

<p>
<a href="#orgd9998d3">The present study has used sufficient time intervals to identify multiple phases of fibre degradation, and subsequently identify the bacterial communities associated with fodder particles in each phase of digestion. Further, this is the first study that has used full-length 16S rRNA gene sequencing to identify the genera and species of bacteria colonising fodder particles in the bovine rumen.</a> <label for="24" class="margin-toggle sidenote-number"></label><input type="checkbox" id="24" class="margin-toggle"/><span class="sidenote">
This sounds like the start of the Discussion
</span>
</p>

<p>
<a href="#org50bed34">Sugarcane bagasse and Lucerne hay have significantly different chemical compositions. Lucerne hay contains 50 &#x2013; 70% non-lignocellulosic components including proteins, non-structural carbohydrates, and minerals which are readily digestible by a wide range of rumen microorganisms. Sugarcane bagasse is composed of gt90% lignocellulose and is degraded by a complex microbial community with cellulolytic activity in the rumen (Pinos-Rodríguez et al. 2002, Guilherme et al. 2015, Harrison 2016, Liu et al. 2016, Moraïs and Mizrahi 2019).</a> <label for="25" class="margin-toggle sidenote-number"></label><input type="checkbox" id="25" class="margin-toggle"/><span class="sidenote">
Again, this sounds like background information and may be better placed in the Introduction
</span>
<a href="#org8b04d48">The differences in the chemical composition of sugarcane bagasse and Lucerne hay significantly affected the kinetics of digestion during incubation in the rumen. The kinetics of DM, NDF, and ADF digestion in sugarcane bagasse and Lucerne hay appeared to be multiphasic. The degradation kinetics of sugarcane bagasse by rumen microorganisms consisted of (i) a rapid digestion phase during which a significant increase in DM, NDF, and ADF digestibility was observed, (ii) an intermediate phase during which the degradation rate decreased, and (iii) a slow degradation phase during which the degradation rate decreased significantly and easily accessible nutrients from the fodder has been consumed. Compared with the sugarcane bagasse, the DM, NDF, and ADF digestibility of Lucerne hay consisted of four phases; including (i) a solubilisation phase in which an initial significant DM loss with no significant corresponding NDF and ADF digestibility in Lucerne hay samples occurred, (ii) a rapid digestion phase during which a significant DM, NDF and ADF digestibility occurred, (iii) a slow digestion phase with a significant decrease in the rate of degradation, and (v) a plateau phase during which no significant DM, NDF, and ADF digestibility was occurred</a>. <label for="26" class="margin-toggle sidenote-number"></label><input type="checkbox" id="26" class="margin-toggle"/><span class="sidenote">
No need to go into so much detail here as that was provided in the results section. Just make a quick reference to the fact that there were big differences in digestibility
</span>
</p>

<p>
Previous studies on rice straw, alfalfa hay, and wheat straw have
demonstrated that the degradation kinetics of fodders in the rumen is
multiphasic. The kinetics of fibre degradation in wheat straw, rice
straw, and switchgrass consists of three phases; (i) relatively rapid DM
degradation (~10%) within 0.5 &#x2013; 1 h of incubation, (ii) a latent phase
during which no significant fibre digestion occurred (between 1 h and 4
&#x2013; 6 h), and (iii) a continuous fibre degradation phase between 4 &#x2013; 6 h
and 72 h (Moraïs and Mizrahi 2019). In another study, the degradation
kinetics of rice straw and alfalfa hay by rumen microorganisms was
investigated. Rice straw was rapidly digested after 0.5 h of incubation
in the rumen followed by a degradation phase (0.5 &#x2013; 48 h) during which
DM of rice straw was digested with a relatively constant rate. DM
degradation of alfalfa hay occurred in three phases including (i) an
initial rapid DM digestion (0.5 h of incubation), (ii) a degradation
phase during which NDF and crude protein were mainly digested (6 &#x2013; 16
h), and (iii) a degradation phase in which the rate of DM degradation
decreased (16 &#x2013; 48 h) (Liu et al. 2016). The kinetics of DM, NDF, and
ADF degradation of Lucerne hay was consistent with those observed with
alfalfa hay except for the plateau phase which is likely due to longer
incubation of Lucerne hay (72 h) in the rumen than alfalfa hay (48 h).
The kinetics of DM, NDF, and ADF degradation of sugarcane bagasse did
not include a latent phase which is likely due to the first
post-incubation timepoint of 4 h instead of 0.5 &#x2013; 1 h. In addition, the
kinetics of NDF degradation of sugarcane bagasse consisted of three
degradation phases with different rates of degradation which is likely
because of longer incubation (72 h) in the rumen compared to rice straw
(48 h). <a href="#orge04bb1d">While the kinetics of degradation of sugarcane bagasse and Lucerne hay in the rumen were consistent with those obtained with rice, barley, and wheat straws, alfalfa hay, and switchgrass, a large number of incubation timepoints used in the present study has provided additional phases of digestion for sugarcane bagasse and Lucerne hay.</a> <label for="27" class="margin-toggle sidenote-number"></label><input type="checkbox" id="27" class="margin-toggle"/><span class="sidenote">
Extra timepoints analysed in this study provided novel insight into phases of digestion that had not previously been observed. This is the key takeaway point and I'm not sure the detail provided before this is necessary to include. Just point the reader to the relevant references
</span>
</p>

<p>
The present study aimed to understand the relationship between the
degradation phases and dynamics of bacterial communities colonising
sugarcane bagasse and Lucerne hay in the bovine rumen. Previous studies
have demonstrated that bacterial communities colonising fodder particles
in the rumen are affected by incubation time and the temporal changes in
the adherent microbial communities are associated with DM digestibility
of fodders (Liu et al. 2016, Cheng et al. 2017, Jin et al. 2018). The
bacterial communities on rice straw separated into two clusters based on
incubation time; including (i) 0.5 h and 6 h, and (ii) 24 h and 48 h.
Another study on barley straw and corn stover demonstrated that
bacterial communities colonising barley straw and corn stover at 48 h of
incubation were distinct from those at 2 &#x2013; 8 h of incubation. The
bacterial communities on rice straw and alfalfa hay were shifted
significantly after 0.5 h of incubation in the rumen and after 16 &#x2013; 48
h compared to 0.5 h. It has been suggested that the two degradation
phases of fodders might correspond to (i) a first cluster of bacterial
community that are involved in degradation of accessible amorphous
regions of the fibre, and (ii) a secondary bacterial group that have
potential crystalline cellulose-degrading capability (Moraïs and Mizrahi
2019). <a href="#orgc99c6f1">The present study confirms that incubation time has a significant impact on the fodder-associated bacterial communities, and there is a relationship between the DM, NDF, and ADF digestibility of sugarcane bagasse and Lucerne hay particles and dynamics of bacterial communities colonising sugarcane bagasse and Lucerne hay in the bovine rumen.</a> <label for="28" class="margin-toggle sidenote-number"></label><input type="checkbox" id="28" class="margin-toggle"/><span class="sidenote">
This is the key point and just use appropriate references to confirm that your findings are similar to previous ones. No need for all the detail
</span>
</p>

<p>
<a href="#org0ac217d">The initial rapid degradation phase of sugarcane bagasse was accompanied by bacterial taxa that play a significant role in plant fibre degradation. <i>Ruminococcus</i> spp. are well-known for their ability to metabolise cellulose and hemicellulose by secreting cellulases, hemicellulases, and other oligosaccharide-degrading enzymes, and produce succinate, formate, and acetate (Flint et al. 2008, Khatoon et al. 2021). <i>Prevotella</i> is one of the most abundant genera in the rumen which has an important role in lignocellulose degradation. Members of <i>Prevotella</i> are significantly involved in the metabolism of starch, cellulose, xylan, pectin, and crude protein and produce acetate, propionate, and succinate (Jin et al. 2018, Zhu et al. 2021). <i>Christensenellaceae R-7 group</i> were consistently abundant on the surface of sugarcane bagasse during the incubation in the rumen; however, their role in the rumen is still unknown (Yang et al. 2020). <i>Rikenellaceae RC9 gut group</i> spp. are found to be involved in the degradation of structural carbohydrates (Zhou et al. 2021). The adhesion of <i>Prevotella</i>, <i>Ruminococcus</i>, and unclassified <i>Rikenellaceae</i> has previously been reported on the surface of rice straw and alfalfa hay (Liu et al. 2016).</a> <label for="29" class="margin-toggle sidenote-number"></label><input type="checkbox" id="29" class="margin-toggle"/><span class="sidenote">
This paragraph basically says that your findings are in agreement with what is already known. That's fine. But is there anything about your particular dataset that is novel? E.g. any novel species-level observation that made use of long Pacbio reads?
</span>
</p>

<p>
<a href="#org24560cb">The bacterial communities adherent to sugarcane bagasse particles in the intermediate degradation phase were significantly less diverse and were distinct compared to those in the initial degradation phase as demonstrated by the alpha and beta diversity, respectively. A significant decrease in the rate of NDF and ADF degradation of sugarcane bagasse in the second degradation phase was accompanied by a significant decrease in the abundance of <i>Prevotella 1</i> and <i>Ruminococcus 1</i>, and a significant increase in the relative abundance of <i>Fibrobacter</i>, <i>Rikenellaceae RC9</i> gut group, <i>Saccharofermentans</i>, and <i>Treponema 2</i>. <i>Fibrobacter</i>, a well-studied rumen bacteria, has high activity against crystalline cellulose and is able to solubilise complex plant cell wall polysaccharides (Terry et al. 2020). It is reported that <i>Treponema</i> and <i>Fibrobacter</i> synergistically work together; <i>Treponema</i> is likely benefiting from the cross-feeding network created by <i>Fibrobacter</i> (Xie et al. 2018). The increased abundance of <i>Fibrobacter</i> and <i>Rikenellaceae RC9 gut group</i> during the intermediate digestion phase may reflect the increased accessibility of cellulose occurred as a result of removal of cell wall matrix earlier in the rapid degradation phase. Previous studies have reported the prevalence of <i>Fibrobacter</i> and <i>Treponema</i> on rice straw, alfalfa, corn stover, barley straw, wheat straw, and switchgrass in the secondary colonising bacterial community (after 16 - 48 h of incubation) (Piao et al. 2014, Liu et al. 2016, Jin et al. 2018, Terry et al. 2020). The genus <i>Saccharofermentans</i> has only one known species which is unable to degrade cellulose but can ferment glucose, sucrose, fructose, cellobiose, starch and produce acetate as the main end-product, and their abundance has been previously reported on rice straw, alfalfa hay, wheat straw, and perennial ryegrass (Huws et al. 2016, Liu et al. 2016, Cheng et al. 2017, Jin et al. 2018, Dai et al. 2021). The increased abundance of this genus might be due to utilisation of released sugars from cellulose degradation. The decrease in diversity and observed shift in bacterial taxa in the intermediate degradation phase reveals that bacterial specialists that can degrade recalcitrant structural carbohydrates become more abundant. These observations are consistent with rumen-incubated alfalfa hay and rice straw samples; Bacteria with more fibre-degrading role such as <i>Fibrobacter</i>, <i>Treponema</i>, unclassified <i>Bacteroidales</i>, and unclassified <i>Rikenellaceae</i> were enriched during the second degradation phase (16 &#x2013; 48 h) (Liu et al. 2016).</a> <label for="30" class="margin-toggle sidenote-number"></label><input type="checkbox" id="30" class="margin-toggle"/><span class="sidenote">
Again, you are highlighting similarities with other studies here. Are there any differences/novel insights?
</span>
</p>

<p>
<a href="#org1fd3049">The slow degradation phase of sugarcane bagasse was accompanied by a significant decrease in the relative abundance of <i>Prevotella</i>, <i>Fibrobacter</i>, and <i>Ruminococcus 1</i>. Previous study has revealed that the abundance of unclassified <i>Rikenellaceae</i> and unclassified <i>Ruminococcaceae</i> has a positive correlation with NDF content and consequently these bacteria play a significant role in the degradation of low-quality fodders such as rice straw (Liu et al. 2016). The abundance of <i>Rikenellaceae RC9</i> gut group, <i>Saccharofermentans</i>, and <i>Treponema 2</i> in the slow degradation phase might be due to utilisation of degradation metabolites. The dynamic changes in the structure and chemical composition of fodders during incubation might change the input and output metabolites and create niche modification that subsequently can cause shifts in microbial communities.</a> <label for="31" class="margin-toggle sidenote-number"></label><input type="checkbox" id="31" class="margin-toggle"/><span class="sidenote">
I dont really follow this
</span>
</p>

<p>
<a href="#orgcc087a7">** Lucerne hay subsection</a> <label for="32" class="margin-toggle sidenote-number"></label><input type="checkbox" id="32" class="margin-toggle"/><span class="sidenote">
Perhaps start a new subsection for lucerne hay. Likewise, make a new sub-section for the sugarcane bagasse section above
</span>
There was no significant NDF and ADF digestion during the solubilisation phase of Lucerne hay&gt;&gt; and <i>Christensenellaceae R-7 group</i>, <i>Butyrivibrio 2</i>, <i>Lachnospiraceae NK3A20 group</i>, <i>Lachnospiraceae XPB1014 group</i>, and <i>Prevotella 1</i> were the most abundant bacterial genera in the microbial community on the surface of Lucerne hay particles. <i>Butyrivibrio</i> is the main butyrate-producing bacteria in the rumen and is known to have proteolytic and oligosaccharide/polysaccharide-degrading activity (Stewart et al. 1997, Grilli et al. 2016, Jin et al. 2018, Dai et al. 2021). The abundance of <i>Prevotella 1</i> spp. on the surface of Lucerne hay is likely due to their ability to degrade oligosaccharide, hemicellulose, and protein (Cheng et al. 2017, Terry et al. 2020). Given that ~40% of DM was digested during the solubilisation phase of Lucerne hay and no significant NDF and ADF digestion were observed, the abundance of these bacterial genera is likely due to the utilisation of soluble sugars, proteins, and minerals. <a href="#orgd06f109">The adhesion of <i>Prevotella 1</i>, unclassified <i>Christensenellaceae</i>, <i>Butyrivibrio 2</i>, and unclassified <i>Lachnospiraceae</i> has been previously reported on the surface of alfalfa hay during <i>in situ</i> incubation in the rumen.</a> <label for="33" class="margin-toggle sidenote-number"></label><input type="checkbox" id="33" class="margin-toggle"/><span class="sidenote">
More similarities. Any differences?
</span>
</p>

<p>
The bacterial communities on Lucerne hay particles in the rapid
digestion phase had significantly higher α-diversity compared to those
in the solubilisation phase, which was accompanied by an increase in the
relative abundance of <i>Treponema 2</i>, <i>Prevotella 1</i>, and <i>Anaeroplasma.</i>
The increased α-diversity is likely due to availability of a wide range
of nutrients such as soluble sugars, proteins, and structural
carbohydrates (hemicellulose, oligosaccharides, and cellulose). The
increased diversity of bacterial community on Lucerne hay after 8 h of
incubation is consistent with previous study on alfalfa hay that showed
the α-diversity increased at 6 h compared to 2 h of incubation and then
remained unchanged between 6 h and 48 h. The diversity of bacterial
communities on Lucerne hay remained unchanged in the slow degradation
and plateau phases which is in line with NDF and ADF digestibility
results. Members of <i>Anaeroplasma</i> metabolise soluble sugars and starch,
and produce acetic acid as the main product. <i>Treponema</i> spp. can
degrade pectin, cellobiose, sucrose, glucose, and fructose (Liu et al.
2016), and their significant increased abundance between 12 h and 48 h
of incubation is due to the utilisation of released sugars as well as
soluble sugars in Lucerne hay. The adhesion of <i>Treponema</i> and
<i>Anaeroplasma</i> has been reported on alfalfa hay (Liu et al. 2016).
Interestingly, <i>Rikenellaceae RC9 gut group</i> and <i>Fibrobacter</i> were
enriched in the slow digestion and plateau phases compared to the rapid
digestion phase which is likely due to increased accessibility of
carbohydrate polymers such as crystalline cellulose. Previous studies
have demonstrated that <i>Rikenellaceae RC9 gut group</i> has a positive
correlation with NDF content. Increased abundance of this bacterial
genus in the slow degradation and plateau phases of Lucerne hay is
likely because of fibre degradation. <i>Ruminococcus 1</i>, <i>Lachnospiraceae
XPB1014 group</i>, and <i>Prevotella 1</i> decreased significantly in abundance
in the plateau phase which is consistent with DM, NDF, and ADF
digestibility.
</p>

<p>
<a href="#orge1d4fa3">Previous studies have demonstrated that bacterial communities adherent to fodder particles were affected by fodder type (Koike et al. 2014, Liu et al. 2016, Elliott et al. 2018, Terry et al. 2020). The present study confirms that the chemical composition and structure of fodder significantly affect the bacterial communities associated with fodder particles in the rumen. The bacterial communities colonising Lucerne hay had significantly higher α-diversity than sugarcane bagasse which is likely due to the differences in their chemical composition. Bacterial communities on Lucerne hay and sugarcane bagasse had similar diversity during the first 8 h of incubation but lucerne hay had higher diversity than sugarcane bagasse after 12 h which may be due to complexity of lucerne hay fibre when most of the readily-fermentable components are consumed. It has been previously demonstrated that fodder complexity promotes microbial diversity (Matthews et al. 2019). A comparison of the bacterial genera on sugarcane bagasse and Lucerne hay particles in the degradation phases of incubation revealed that bacterial taxa detected in the intermediate phase of sugarcane bagasse and slow digestion and plateau phases of Lucerne hay had more specialised microorganisms with cellulytic role. However, the bacterial community on sugarcane bagasse and Lucerne hay differed in the abundance which might be due to the differences in plant cell wall composition and structure, and/or microbial interaction. A higher abundance of <i>Fibrobacter</i> and <i>Saccharofermentans</i> on sugarcane bagasse is likely because of the low content of soluble and readily digestible carbohydrates, subsequently, fibrolytic bacteria became more dominant. <i>Anaeroplasma</i> and <i>Butyrivibrio 2</i> had a higher abundance in the digestion phase of Lucerne hay which is likely because of the protein, starch, and easily accessible carbohydrates in Lucerne hay. It is reported that <i>Anaeroplasma</i> is more abundant in fodders with high crude protein content (Takizawa et al. 2021).</a> <label for="34" class="margin-toggle sidenote-number"></label><input type="checkbox" id="34" class="margin-toggle"/><span class="sidenote">
Perhaps this can go closer to the start of the Discussion as it summarises the key findings
</span>
</p>

<p>
<a href="#org0537d25">In summary, this study provided a comprehensive overview of the effect of fodder composition on the diversity and composition of bacterial communities colonising sugarcane bagasse and Lucerne hay and demonstrated that the colonising bacterial community is affected by the differences in crude protein and fibre (NDF and ADF) contents. The dynamics of changes in bacterial communities colonising sugarcane bagasse and Lucerne hay during <i>in situ</i> rumen incubation is likely driven by ecological niche partitioning, microbial interactions (<i>e.g.</i> hydrogen transfer), and/or competitions between microorganisms for a mutual resource (Moraïs and Mizrahi 2019, Moraïs and Mizrahi 2019). Carbohydrates within plant cell wall are different in terms of rate of degradation which subsequently causes changes in the colonising microbial community over time after feeding. Further, the changes in the relative abundance of different members of bacterial community is likely because fodder fibre becomes increasingly crystalline during digestion. It might also be due to ecological niche modification, a process during which the metabolic activity of microorganisms modifies their local environment and creates new niches for other microorganisms. It is suggested that a rock&#x2013;paper&#x2013;scissor-type interactions might exist in complex ecosystems such as rumen, whereby despite of negative competitions among microorganisms for mutual substrates and growth factors, none microbe become consistently dominant because microorganisms are embedded in networks with multiple environmental factors that allows species coexistence.</a> <label for="35" class="margin-toggle sidenote-number"></label><input type="checkbox" id="35" class="margin-toggle"/><span class="sidenote">
The summary should focus on how your approach led to novel insights and how these in turn might have practical applications. You should probably also address limitations such as the lack of analysis of fungi etc. As otherwise this will be picked up by the reviewers
</span>
</p>
</div>
</div>

<div id="outline-container-org8469557" class="outline-2">
<h2 id="org8469557"><span class="section-number-2">7</span> <b>5. Conclusion</b></h2>
<div class="outline-text-2" id="text-7">
<p>
The results of this study showed that the kinetics of fodder digestion
and the bacterial communities colonising sugarcane bagasse and Lucerne
hay were affected by incubation time in the rumen and fodder type.
Further, a strong relationship between the kinetics of DM, NDF, and ADF
digestibility and fodder-adherent bacterial communities was observed.
<a href="#org57e6f48">The bacterial community associated with sugarcane bagasse and Lucerne hay and planktonic bacterial community were characterised using full-length PacBio sequencing.</a> <label for="36" class="margin-toggle sidenote-number"></label><input type="checkbox" id="36" class="margin-toggle"/><span class="sidenote">
The benefits of which were?
</span>
The bacterial communities that colonised
sugarcane bagasse and Lucerne hay were distinct between phases of
incubation. The NDF and ADF digestion of bagasse during the digestion
phase was accompanied by bacterial genera with high cellulolytic
activity and the plateau phase was dominated by bacterial genera that
are mostly involved in the utilisation of degradation products from the
digestion phase. A rapid DM loss in Lucerne hay was accompanied by
bacterial genera that utilise simple carbohydrates and crude protein,
and bacterial taxa shifted to fibrolytic bacteria in the digestion and
plateau phases to metabolise structural carbohydrates. A comparison of
bacterial communities associated with sugarcane bagasse and Lucerne hay
demonstrated that chemical composition of fodder significantly affect
the composition of bacterial communities colonising fodder particles.
These findings are important for the improvement of fodder use
efficiency in ruminants.
</p>
</div>
</div>

<div id="outline-container-org3acb8f3" class="outline-2">
<h2 id="org3acb8f3"><span class="section-number-2">8</span> <b>Acknowledgements</b></h2>
<div class="outline-text-2" id="text-8">
<p>
<a href="#org003028c">X</a> <label for="37" class="margin-toggle sidenote-number"></label><input type="checkbox" id="37" class="margin-toggle"/><span class="sidenote">
CARF, UQ Gatton etc
</span>
</p>
</div>
</div>

<div id="outline-container-orgecb18d7" class="outline-2">
<h2 id="orgecb18d7"><span class="section-number-2">9</span> <b>Tables</b></h2>
<div class="outline-text-2" id="text-9">
<p>
Table 1. Dry matter and chemical composition (% dry matter) of sugarcane
bagasse and Lucerne hay
</p>

<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">


<colgroup>
<col  class="org-left" />

<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<thead>
<tr>
<th scope="col" class="org-left"> </th>
<th scope="col" class="org-left"><b>Sugarcane bagasse</b></th>
<th scope="col" class="org-left"><b>Lucerne hay</b></th>
</tr>
</thead>
<tbody>
<tr>
<td class="org-left">Dry matter (%)</td>
<td class="org-left">96.5 ± 0.3</td>
<td class="org-left">92.5 ± 0.3</td>
</tr>

<tr>
<td class="org-left">Neutral detergent fibre (%)</td>
<td class="org-left">88.5 ± 0.2</td>
<td class="org-left">36.7 ± 0.3</td>
</tr>

<tr>
<td class="org-left">Acid detergent fibre (%)</td>
<td class="org-left">57.0 ± 0.2</td>
<td class="org-left">26.6 ± 0.2</td>
</tr>

<tr>
<td class="org-left">Crude protein (%)</td>
<td class="org-left">1.2 ± 0.0</td>
<td class="org-left">20.3 ± 0.1</td>
</tr>

<tr>
<td class="org-left">Ash (%)</td>
<td class="org-left">2.0 ± 0.1</td>
<td class="org-left">9.2 ± 0.5</td>
</tr>

<tr>
<td class="org-left">Cellulose (%DF<sup>1</sup>)</td>
<td class="org-left">35.0 ± 0.4</td>
<td class="org-left">31.6 ± 0.4</td>
</tr>

<tr>
<td class="org-left">Hemicellulose (%DF)</td>
<td class="org-left">19.6 ± 0.2</td>
<td class="org-left">15.2 ± 0.1</td>
</tr>

<tr>
<td class="org-left">Lignin (%DF)</td>
<td class="org-left">28.1 ± 0.5</td>
<td class="org-left">22.4 ± 0.3</td>
</tr>

<tr>
<td class="org-left">Water extractives (%DF)</td>
<td class="org-left">5.6 ± 0.2</td>
<td class="org-left">26.4 ± 0.2</td>
</tr>

<tr>
<td class="org-left">Ethanol extractives (%DF)</td>
<td class="org-left">1.8 ± 0.1</td>
<td class="org-left">3.9 ± 0.1</td>
</tr>
</tbody>
</table>

<p>
^{1} Dry fibre
</p>
</div>
</div>

<div id="outline-container-org0a0a8f9" class="outline-2">
<h2 id="org0a0a8f9"><span class="section-number-2">10</span> <b>Figures</b></h2>
<div class="outline-text-2" id="text-10">
</div>
<div id="outline-container-org9ecb625" class="outline-3">
<h3 id="org9ecb625"><span class="section-number-3">10.1</span> Figure 1</h3>
<div class="outline-text-3" id="text-10-1">

<div id="org4db1e86" class="figure">
<p><img src="media/image4.png" alt="image4.png" />
</p>
</div>
</div>
</div>

<div id="outline-container-orgfc3f850" class="outline-3">
<h3 id="orgfc3f850"><span class="section-number-3">10.2</span> Figure 2</h3>
<div class="outline-text-3" id="text-10-2">

<div id="org59b8b58" class="figure">
<p><img src="media/image5.png" alt="image5.png" />
</p>
</div>
</div>
</div>

<div id="outline-container-orgbf7c5f4" class="outline-3">
<h3 id="orgbf7c5f4"><span class="section-number-3">10.3</span> Figure 3</h3>
<div class="outline-text-3" id="text-10-3">
<p>
<a href="media/image6.tiff">media/image6.tiff</a>
</p>
</div>
</div>

<div id="outline-container-org569925e" class="outline-3">
<h3 id="org569925e"><span class="section-number-3">10.4</span> Figure 4</h3>
<div class="outline-text-3" id="text-10-4">
<p>
<a href="media/image7.tiff">media/image7.tiff</a>
</p>
</div>
</div>

<div id="outline-container-org7b637b9" class="outline-3">
<h3 id="org7b637b9"><span class="section-number-3">10.5</span> Figure 5</h3>
<div class="outline-text-3" id="text-10-5">
<p>
<a href="media/image8.tiff">media/image8.tiff</a>
</p>
</div>
</div>

<div id="outline-container-org5329117" class="outline-3">
<h3 id="org5329117"><span class="section-number-3">10.6</span> Figure 6</h3>
<div class="outline-text-3" id="text-10-6">
<p>
<a href="media/image9.tiff">media/image9.tiff</a>
</p>
</div>
</div>

<div id="outline-container-org23c9ae9" class="outline-3">
<h3 id="org23c9ae9"><span class="section-number-3">10.7</span> Figure 7.</h3>
<div class="outline-text-3" id="text-10-7">
<p>
*<br />
</p>
<p>
*
</p>
</div>
</div>
</div>

<div id="outline-container-orge2966b0" class="outline-2">
<h2 id="orge2966b0"><span class="section-number-2">11</span> <b>Supplementary figures</b></h2>
<div class="outline-text-2" id="text-11">

<div id="org11eba8f" class="figure">
<p><img src="media/image10.png" alt="image10.png" />
</p>
</div>

<p>
Figure S1.
</p>

<p>
<a href="media/image11.tiff">media/image11.tiff</a>
</p>

<p>
Figure S2.
</p>


<div id="org7160a6b" class="figure">
<p><img src="media/image12.png" alt="image12.png" />
</p>
</div>

<p>
Figure S3
</p>


<div id="org4b2c944" class="figure">
<p><img src="media/image13.png" alt="image13.png" />
</p>
</div>

<p>
Figure S4.
</p>

<p>
<a href="media/image14.tiff">media/image14.tiff</a>
</p>

<p>
Figure S5.
</p>
</div>
</div>

<div id="outline-container-org26b4db1" class="outline-2">
<h2 id="org26b4db1"><span class="section-number-2">12</span> Figure Legends</h2>
<div class="outline-text-2" id="text-12">
</div>
<div id="outline-container-org43bde79" class="outline-3">
<h3 id="org43bde79"><span class="section-number-3">12.1</span> Figure 1. DM (%), NDF (%), and ADF (%) digestibility of sugarcane</h3>
<div class="outline-text-3" id="text-12-1">
<p>
bagasse and Lucerne hay after 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h
<i>in situ</i> bovine rumen incubation
</p>
</div>
</div>

<div id="outline-container-orgc8926e3" class="outline-3">
<h3 id="orgc8926e3"><span class="section-number-3">12.2</span> Figure 2. The alpha diversity of (A) overall bacterial community</h3>
<div class="outline-text-3" id="text-12-2">
<p>
adherent to sugarcane bagasse and Lucerne hay during incubation in the
rumen, (B) bacterial community adherent to sugarcane bagasse and Lucerne
hay after 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h, and (C) bacterial
community adherent to sugarcane bagasse and Lucerne hay in the phases of
digestion.
</p>
</div>
</div>

<div id="outline-container-org3611327" class="outline-3">
<h3 id="org3611327"><span class="section-number-3">12.3</span> Figure 3. Principal coordinates analysis (PCoA) plot of the beta</h3>
<div class="outline-text-3" id="text-12-3">
<p>
diversity of (A) bacterial community adherent to sugarcane bagasse after
4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h (B) bacterial community
adherent to Lucerne hay after 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h
(C) bacterial community adherent to sugarcane bagasse between phases of
incubation, and (D) bacterial community adherent to Lucerne hay between
phases of incubation
</p>
</div>
</div>

<div id="outline-container-org81d6345" class="outline-3">
<h3 id="org81d6345"><span class="section-number-3">12.4</span> Figure 4. Heatmap of relative abundance of bacterial phyla adherent to</h3>
<div class="outline-text-3" id="text-12-4">
<p>
sugarcane bagasse and Lucerne hay identified after 4, 8, 12, 18, 24, 32,
40, 48, 60, and 72 h <i>in</i> <i>situ</i> bovine rumen incubation using
full-length 16S rRNA PacBio sequencing. The top 20 bacterial taxa are
shown.
</p>
</div>
</div>

<div id="outline-container-org69cb1c2" class="outline-3">
<h3 id="org69cb1c2"><span class="section-number-3">12.5</span> Figure 5. Heatmap of relative abundance of bacterial genera adherent to</h3>
<div class="outline-text-3" id="text-12-5">
<p>
sugarcane bagasse and Lucerne hay identified after 4, 8, 12, 18, 24, 32,
40, 48, 60, and 72 h <i>situ</i> bovine rumen incubation using full-length
16S rRNA PacBio sequencing. The top 20 bacterial taxa are shown.
</p>
</div>
</div>

<div id="outline-container-org0d3a20a" class="outline-3">
<h3 id="org0d3a20a"><span class="section-number-3">12.6</span> Figure 6. Heatmap of the relative distribution of bacterial genera in</h3>
<div class="outline-text-3" id="text-12-6">
<p>
the digestion phases of <i>in situ</i> rumen incubation of sugarcane bagasse
and Lucerne hay.
</p>
</div>
</div>

<div id="outline-container-orgd078eff" class="outline-3">
<h3 id="orgd078eff"><span class="section-number-3">12.7</span> Figure 7. Heatmap of relative abundance of planktonic bacterial genera</h3>
<div class="outline-text-3" id="text-12-7">
<p>
identified at 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h incubation of
sugarcane bagasse and Lucerne hay using full-length 16S rRNA PacBio
sequencing. The top 20 bacterial taxa are shown.
</p>
</div>
</div>

<div id="outline-container-orgf18a509" class="outline-3">
<h3 id="orgf18a509"><span class="section-number-3">12.8</span> Figure S1. The correlation between alpha diversity of bacterial</h3>
<div class="outline-text-3" id="text-12-8">
<p>
community adherent to (A) sugarcane bagasse (B) Lucerne hay and NDF
digestibility during bovine rumen incubation
</p>
</div>
</div>

<div id="outline-container-org7069e9b" class="outline-3">
<h3 id="org7069e9b"><span class="section-number-3">12.9</span> Figure S2. A principal coordinates analysis (PCoA) plot of the beta</h3>
<div class="outline-text-3" id="text-12-9">
<p>
diversity of overall bacterial communities associated with sugarcane
bagasse and Lucerne hay during incubation in the rumen.
</p>
</div>
</div>

<div id="outline-container-org0a8ae2f" class="outline-3">
<h3 id="org0a8ae2f"><span class="section-number-3">12.10</span> Figure S3. The epiphytic bacterial community adherent to sugarcane</h3>
<div class="outline-text-3" id="text-12-10">
<p>
bagasse and Lucerne hay at the (A) phylum, and (B) genus levels.
</p>
</div>
</div>

<div id="outline-container-org5558e3d" class="outline-3">
<h3 id="org5558e3d"><span class="section-number-3">12.11</span> Figure S4. Statistically significant differences in the bacterial (A)</h3>
<div class="outline-text-3" id="text-12-11">
<p>
phyla, (B) genera colonising sugarcane bagasse and Lucerne hay during
incubation in the rumen. Significantly different taxa (q value &lt; 0.05)
are coloured red. The horizontal dashed line indicates q value of 0.05.
</p>
</div>
</div>

<div id="outline-container-org1237ced" class="outline-3">
<h3 id="org1237ced"><span class="section-number-3">12.12</span> Figure S5. Heatmap of relative abundance of planktonic bacterial phyla</h3>
<div class="outline-text-3" id="text-12-12">
<p>
identified at 4, 8, 12, 18, 24, 32, 40, 48, 60, and 72 h incubation of
sugarcane bagasse and Lucerne hay.
</p>
</div>
</div>
</div>

<div id="outline-container-orgf8dae60" class="outline-2">
<h2 id="orgf8dae60"><span class="section-number-2">13</span> References</h2>
<div class="outline-text-2" id="text-13">
<p>
Ahmadi F, Zamiri MJ, Khorvash M/, et al./ Pre-treatment of sugarcane
bagasse with a combination of sodium hydroxide and lime for improving
the ruminal degradability: optimization of process parameters using
response surface methodology. <i>J Appl Anim Res</i>. 2016;44:287-96.
</p>

<p>
Andersen KS, Kirkegaard RH, Karst SM/, et al./ ampvis2: an R package to
analyse and visualise 16S rRNA amplicon data. <i>BioRxiv</i>. 2018;299537.
</p>

<p>
Andrews S (2010) FastQC: a quality control tool for high throughput
sequence data. Babraham Bioinformatics, Babraham Institute, Cambridge,
United Kingdom.
</p>

<p>
Bolyen E, Rideout JR, Dillon MR/, et al./ Reproducible, interactive,
scalable and extensible microbiome data science using QIIME 2. <i>Nature
biotechnology</i>. 2019;37:852-57.
</p>

<p>
Callahan BJ, McMurdie PJ, Rosen MJ/, et al./ DADA2: High-resolution
sample inference from Illumina amplicon data. <i>Nature methods</i>.
2016;13:581-83.
</p>

<p>
Cammack KM, Austin KJ, Lamberson WR/, et al./ RUMINANT NUTRITION
SYMPOSIUM: Tiny but mighty: the role of the rumen microbes in livestock
production. <i>J Anim Sci</i>. 2018;96:752-70.
</p>

<p>
Cheng Y, Wang Y, Li Y/, et al./ Progressive colonization of bacteria and
degradation of rice straw in the rumen by Illumina sequencing. <i>Front
Microbiol</i>. 2017;8:2165.
</p>

<p>
Core RT. R: A language and environment for statistical computing. 2013.
</p>

<p>
Dai Q, Ma J, Cao G/, et al./ Comparative study of growth performance,
nutrient digestibility, and ruminal and fecal bacterial community
between yaks and cattle-yaks raised by stall-feeding. <i>AMB Express</i>.
2021;11:1-11.
</p>

<p>
DeSantis TZ, Hugenholtz P, Larsen N/, et al./ Greengenes, a
chimera-checked 16S rRNA gene database and workbench compatible with
ARB. <i>Applied and environmental microbiology</i>. 2006;72:5069-72.
</p>

<p>
Dixon P. VEGAN, a package of R functions for community ecology. <i>J Veg
Sci</i>. 2003;14:927-30.
</p>

<p>
Du C, Nan X, Wang K/, et al./ Evaluation of the digestibility of
steam-exploded wheat straw by ruminal fermentation, sugar yield and
microbial structure in vitro. <i>RSC Advances</i>. 2019;9:41775-82.
</p>

<p>
Elliott CL, Edwards JE, Wilkinson TJ/, et al./ Using 'Omic Approaches to
Compare Temporal Bacterial Colonization of Lolium perenne, Lotus
corniculatus, and Trifolium pratense in the Rumen. <i>Front Microbiol</i>.
2018;9:2184.
</p>

<p>
Firkins JL. Invited Review: Advances in rumen efficiency**Presented as
part of the ARPAS Symposium: New Advances in Dairy Efficiency at the
American Dairy Science Association Virtual Annual Meeting, June 2020.
<i>Applied Animal Science</i>. 2021;37:388-403.
</p>

<p>
Flint HJ, Bayer EA, Rincon MT/, et al./ Polysaccharide utilization by
gut bacteria: potential for new insights from genomic analysis. <i>Nat Rev
Microbiol</i>. 2008;6:121-31.
</p>

<p>
Grilli DJ, Fliegerová K, Kopečný J/, et al./ Analysis of the rumen
bacterial diversity of goats during shift from forage to concentrate
diet. <i>Anaerobe</i>. 2016;42:17-26.
</p>

<p>
Guilherme A, Dantas P, Santos E/, et al./ Evaluation of composition,
characterization and enzymatic hydrolysis of pretreated sugar cane
bagasse. <i>Braz J Chem Eng</i>. 2015;32:23-33.
</p>

<p>
Halmemies-Beauchet-Filleau A, Rinne M, Lamminen M/, et al./ Alternative
and novel feeds for ruminants: nutritive value, product quality and
environmental aspects. <i>Animal</i>. 2018;12:295-309.
</p>

<p>
Harrison MD. Sugarcane-derived animal feed. <i>Sugarcane-based biofuels
and bioproducts,</i> Wiley,2016,281-300.
</p>

<p>
Huws SA, Edwards JE, Creevey CJ/, et al./ Temporal dynamics of the
metabolically active rumen bacteria colonizing fresh perennial ryegrass.
<i>FEMS Microbiol Ecol</i>. 2016;92:fiv137.
</p>

<p>
Jin W, Wang Y, Li Y/, et al./ Temporal changes of the bacterial
community colonizing wheat straw in the cow rumen. <i>Anaerobe</i>.
2018;50:1-8.
</p>

<p>
Khatoon M, Patel SH, Pandit RJ/, et al./ Rumen and fecal microbial
profiles in cattle fed high lignin diets using metagenome analysis.
<i>Anaerobe</i>. 2021;102508.
</p>

<p>
Koike S, Yabuki H &amp; Kobayashi Y. Interaction of rumen bacteria as
assumed by colonization patterns on untreated and alkali-treated rice
straw. <i>Anim Sci J</i>. 2014;85:524-31.
</p>

<p>
Lengowski MB, Witzig M, Möhring J/, et al./ Effects of corn silage and
grass silage in ruminant rations on diurnal changes of microbial
populations in the rumen of dairy cows. <i>Anaerobe</i>. 2016;42:6-16.
</p>

<p>
Liu J, Zhang M, Xue C/, et al./ Characterization and comparison of the
temporal dynamics of ruminal bacterial microbiota colonizing rice straw
and alfalfa hay within ruminants. <i>J Dairy Sci</i>. 2016;99:9668-81.
</p>

<p>
Mandal S, Van Treuren W, White RA/, et al./ Analysis of composition of
microbiomes: a novel method for studying microbial composition.
<i>Microbial ecology in health and disease</i>. 2015;26:27663.
</p>

<p>
Martin M. Cutadapt removes adapter sequences from high-throughput
sequencing reads. <i>EMBnet journal</i>. 2011;17:10-12.
</p>

<p>
Matthews C, Crispie F, Lewis E/, et al./ The rumen microbiome: a crucial
consideration when optimising milk and meat production and nitrogen
utilisation efficiency. <i>Gut Microbes</i>. 2019;10:115-32.
</p>

<p>
McGrath J, Duval SM, Tamassia LF/, et al./ Nutritional strategies in
ruminants: A lifetime approach. <i>Research in Veterinary Science</i>.
2018;116:28-39.
</p>

<p>
Mizrahi I &amp; Jami E. Review: The compositional variation of the rumen
microbiome and its effect on host performance and methane emission.
<i>Animal</i>. 2018;12:s220-s32.
</p>

<p>
Mizrahi I, Wallace RJ &amp; Moraïs S. The rumen microbiome: balancing food
security and environmental impacts. <i>Nat Rev Microbiol</i>. 2021;19:553-66.
</p>

<p>
Moraïs S &amp; Mizrahi I. Islands in the stream: from individual to communal
fiber degradation in the rumen ecosystem. <i>FEMS Microbiol Rev</i>.
2019;43:362-79.
</p>

<p>
Moraïs S &amp; Mizrahi I. The Road Not Taken: The Rumen Microbiome,
Functional Groups, and Community States. <i>Trends Microbiol</i>.
2019;27:538-49.
</p>

<p>
Ogle D, Doll J, Wheeler P/, et al./ (2021) Simple Fisheries Stock
Assessment Methods. .
</p>

<p>
Piao H, Lachman M, Malfatti S/, et al./ Temporal dynamics of fibrolytic
and methanogenic rumen microorganisms during in situ incubation of
switchgrass determined by 16S rRNA gene profiling. <i>Front Microbiol</i>.
2014;5.
</p>

<p>
Pinos-Rodríguez JM, González SS, Mendoza GD/, et al./ Effect of
exogenous fibrolytic enzyme on ruminal fermentation and digestibility of
alfalfa and rye-grass hay fed to lambs. <i>J Anim Sci</i>. 2002;80:3016-20.
</p>

<p>
Quast C, Pruesse E, Yilmaz P/, et al./ The SILVA ribosomal RNA gene
database project: improved data processing and web-based tools. <i>Nucleic
acids research</i>. 2012;41:D590-D96.
</p>

<p>
Shannon CE &amp; Weaver W. <i>The mathematical theory of communication</i>.
University of Illinois Press, Champaign, IL, US,1949.
</p>

<p>
Sluiter A, Hames B, Ruiz R/, et al./ Determination of structural
carbohydrates and lignin in biomass. <i>Laboratory analytical procedure</i>.
2008;1617:1-16.
</p>

<p>
Snelling TJ, Auffret MD, Duthie C-A/, et al./ Temporal stability of the
rumen microbiota in beef cattle, and response to diet and supplements.
<i>Anim Microbiome</i>. 2019;1:16.
</p>

<p>
So S, Cherdthong A, Wanapat M/, et al./ Fermented sugarcane bagasse with
Lactobacillus combined with cellulase and molasses promotes in vitro gas
kinetics, degradability, and ruminal fermentation patterns compared to
rice straw. <i>Anim Biotechnol</i>. 2020;1-12.
</p>

<p>
Stewart C, Flint H &amp; Bryant M. The rumen bacteria. <i>The rumen microbial
ecosystem,</i> Springer,1997,10-72.
</p>

<p>
Straub D, Blackwell N, Langarica-Fuentes A/, et al./ Interpretations of
Environmental Microbial Community Studies Are Biased by the Selected 16S
rRNA (Gene) Amplicon Sequencing Pipeline. <i>Front Microbiol</i>. 2020;11.
</p>

<p>
Takizawa S, Asano R, Fukuda Y/, et al./ Characteristics of various
fibrolytic isozyme activities in the rumen microbial communities of
Japanese Black and Holstein Friesian cattle under different conditions.
<i>Anim Sci J</i>. 2021;92:e13653.
</p>

<p>
Terry SA, Ribeiro GO, Conrad CC/, et al./ Pretreatment of crop residues
by ammonia fiber expansion (AFEX) alters the temporal colonization of
feed in the rumen by rumen microbes. <i>FEMS Microbiol Ecol</i>.
2020;96:fiaa074.
</p>

<p>
Vitti DMSS, Abdalla AL, Silva Filho JC/, et al./ Misleading
relationships between in situ rumen dry matter disappearance, chemical
analyses and in vitro gas production and digestibility, of sugarcane
bagasse treated with varying levels of electron irradiation and ammonia.
<i>Anim Feed Sci Technol</i>. 1999;79:145-53.
</p>

<p>
Wuertz D, Setz T, Chalabi Y/, et al./ Rmetrics - Markets and Basic
Statistics. 2020.
</p>

<p>
Xie X, Yang C, Guan LL/, et al./ Persistence of cellulolytic bacteria
<i>fibrobacter</i> and <i>treponema</i> after short-term corn stover-based dietary
intervention reveals the potential to improve rumen fibrolytic function.
<i>Front Microbiol</i>. 2018;9:1363.
</p>

<p>
Yang C, Tsedan G, Liu Y/, et al./ Shrub coverage alters the rumen
bacterial community of yaks (Bos grunniens) grazing in alpine meadows.
<i>Journal of Animal Science and Technology</i>. 2020;62:504.
</p>

<p>
Zhou Y, Sun L, Cheng Q/, et al./ Effect of pelleted alfalfa or native
grass total mixed ration on the rumen bacterial community and growth
performance of lambs on the Mongolian Plateau. <i>Small Ruminant
Research</i>. 2021;106610.
</p>

<p>
Zhu Z, Difford GF, Noel SJ/, et al./ Stability Assessment of the Rumen
Bacterial and Archaeal Communities in Dairy Cows Within a Single
Lactation and Its Association With Host Phenotype. <i>Front Microbiol</i>.
2021;12:601.
</p>
</div>
</div>

<div id="outline-container-orgd337abb" class="outline-2">
<h2 id="orgd337abb"><span class="section-number-2">14</span> KJD Comments</h2>
<div class="outline-text-2" id="text-14">
</div>
<div id="outline-container-orgf0612e2" class="outline-3">
<h3 id="orgf0612e2"><span class="section-number-3">14.1</span> Corrections</h3>
<div class="outline-text-3" id="text-14-1">
</div>
<div id="outline-container-orgeee1fc7" class="outline-4">
<h4 id="orgeee1fc7"><span class="section-number-4">14.1.1</span> Kevin Dudley ^{4}</h4>
</div>
<div id="outline-container-org4e39cb5" class="outline-4">
<h4 id="org4e39cb5"><span class="section-number-4">14.1.2</span> 60% to 70%</h4>
</div>
<div id="outline-container-org7a1332d" class="outline-4">
<h4 id="org7a1332d"><span class="section-number-4">14.1.3</span> stomach compartment in</h4>
</div>
<div id="outline-container-org5402742" class="outline-4">
<h4 id="org5402742"><span class="section-number-4">14.1.4</span> DM</h4>
</div>
<div id="outline-container-org515da96" class="outline-4">
<h4 id="org515da96"><span class="section-number-4">14.1.5</span> Microbial colonisation of fodder particles in the rumen</h4>
</div>
<div id="outline-container-orgd62f450" class="outline-4">
<h4 id="orgd62f450"><span class="section-number-4">14.1.6</span> 72 h of incubation</h4>
</div>
<div id="outline-container-org1e72437" class="outline-4">
<h4 id="org1e72437"><span class="section-number-4">14.1.7</span> data not shown in</h4>
</div>
<div id="outline-container-org8a09982" class="outline-4">
<h4 id="org8a09982"><span class="section-number-4">14.1.8</span> The proportion of Fibrobacteres was significantly higher in sugarcane bagasse than in Lucerne hay samples at 24 h of incubation, and Lucerne hay had a higher abundance of Tenericutes at 8, 12, and 18 h of incubation than sugarcane bagasse</h4>
</div>
</div>
<div id="outline-container-orga210deb" class="outline-3">
<h3 id="orga210deb"><span class="section-number-3">14.2</span> Comments</h3>
<div class="outline-text-3" id="text-14-2">
</div>
<ol class="org-ol">
<li><a id="orgfeadd50"></a>KJD1<br />
<div class="outline-text-5" id="text-14-2-0-1">
</div>
</li>
<li><a id="orgca2744e"></a>KJD2<br />
<div class="outline-text-5" id="text-14-2-0-2">
</div>
</li>
<li><a id="orgba9300a"></a>KJD3<br />
<div class="outline-text-5" id="text-14-2-0-3">
</div>
</li>
<li><a id="org30a74d3"></a>KJD4<br />
<div class="outline-text-5" id="text-14-2-0-4">
</div>
</li>
<li><a id="orgb16cebd"></a>KJD5<br />
<div class="outline-text-5" id="text-14-2-0-5">
</div>
</li>
<li><a id="org5dfa9ef"></a>KJD6<br />
<div class="outline-text-5" id="text-14-2-0-6">
</div>
</li>
<li><a id="org4e97962"></a>KJD7<br />
<div class="outline-text-5" id="text-14-2-0-7">
</div>
</li>
<li><a id="org8bfcf53"></a>KJD8<br />
<div class="outline-text-5" id="text-14-2-0-8">
</div>
</li>
<li><a id="orgc7b8142"></a>KJD9<br />
<div class="outline-text-5" id="text-14-2-0-9">
</div>
</li>
<li><a id="org402d330"></a>KJD10<br />
<div class="outline-text-5" id="text-14-2-0-10">
</div>
</li>
<li><a id="org05ca604"></a>KJD11<br />
<div class="outline-text-5" id="text-14-2-0-11">
</div>
</li>
<li><a id="orgf94aaff"></a>KJD12<br />
<div class="outline-text-5" id="text-14-2-0-12">
</div>
</li>
<li><a id="org3a81e63"></a>KJD13<br />
<div class="outline-text-5" id="text-14-2-0-13">
</div>
</li>
<li><a id="orgd9c1d98"></a>KJD14<br />
<div class="outline-text-5" id="text-14-2-0-14">
</div>
</li>
<li><a id="org0390f73"></a>KJD15<br />
<div class="outline-text-5" id="text-14-2-0-15">
</div>
</li>
<li><a id="orgf0383dd"></a>KJD16<br />
<div class="outline-text-5" id="text-14-2-0-16">
</div>
</li>
<li><a id="orgee97a60"></a>KJD17<br />
<div class="outline-text-5" id="text-14-2-0-17">
</div>
</li>
<li><a id="org0e445c6"></a>KJD18<br />
<div class="outline-text-5" id="text-14-2-0-18">
</div>
</li>
<li><a id="orgd1be9b5"></a>KJD19<br />
<div class="outline-text-5" id="text-14-2-0-19">
</div>
</li>
<li><a id="orgc5ccd89"></a>KJD20<br />
<div class="outline-text-5" id="text-14-2-0-20">
</div>
</li>
<li><a id="org2c35723"></a>KJD21<br />
<div class="outline-text-5" id="text-14-2-0-21">
</div>
</li>
<li><a id="org4836c7c"></a>KJD22<br />
<div class="outline-text-5" id="text-14-2-0-22">
</div>
</li>
<li><a id="org3745475"></a>KJD23<br />
<div class="outline-text-5" id="text-14-2-0-23">
</div>
</li>
<li><a id="orgd9998d3"></a>KJD24<br />
<div class="outline-text-5" id="text-14-2-0-24">
</div>
</li>
<li><a id="org50bed34"></a>KJD25<br />
<div class="outline-text-5" id="text-14-2-0-25">
</div>
</li>
<li><a id="org8b04d48"></a>KJD26<br />
<div class="outline-text-5" id="text-14-2-0-26">
</div>
</li>
<li><a id="orge04bb1d"></a>KJD27<br />
<div class="outline-text-5" id="text-14-2-0-27">
</div>
</li>
<li><a id="orgc99c6f1"></a>KJD28<br />
<div class="outline-text-5" id="text-14-2-0-28">
</div>
</li>
<li><a id="org0ac217d"></a>KJD29<br />
<div class="outline-text-5" id="text-14-2-0-29">
</div>
</li>
<li><a id="org24560cb"></a>KJD30<br />
<div class="outline-text-5" id="text-14-2-0-30">
</div>
</li>
<li><a id="org1fd3049"></a>KJD31<br />
<div class="outline-text-5" id="text-14-2-0-31">
</div>
</li>
<li><a id="orgcc087a7"></a>KJD32<br />
<div class="outline-text-5" id="text-14-2-0-32">
</div>
</li>
<li><a id="orgd06f109"></a>KJD33<br />
<div class="outline-text-5" id="text-14-2-0-33">
</div>
</li>
<li><a id="orge1d4fa3"></a>KJD34<br />
<div class="outline-text-5" id="text-14-2-0-34">
</div>
</li>
<li><a id="org0537d25"></a>KJD35<br />
<div class="outline-text-5" id="text-14-2-0-35">
</div>
</li>
<li><a id="org57e6f48"></a>KJD36<br />
<div class="outline-text-5" id="text-14-2-0-36">
</div>
</li>
<li><a id="org003028c"></a>KJD37<br />
<div class="outline-text-5" id="text-14-2-0-37">
</div>
</li>
</ol>
</div>
</div>
<!-- Footnotes --><!-- 
<div class="footdef"><sup><a id="fn.1" class="footnum" href="#fnr.1">1</a></sup> <div class="footpara"><p class="footpara">
Can this sentence be simplified?
</p></div></div>

<div class="footdef"><sup><a id="fn.2" class="footnum" href="#fnr.2">2</a></sup> <div class="footpara"><p class="footpara">
Is rumen fermentation the correct terminology?
</p></div></div>

<div class="footdef"><sup><a id="fn.3" class="footnum" href="#fnr.3">3</a></sup> <div class="footpara"><p class="footpara">
Can this section be shortened by referring to previous papers?
</p></div></div>

<div class="footdef"><sup><a id="fn.4" class="footnum" href="#fnr.4">4</a></sup> <div class="footpara"><p class="footpara">
Is this kit suitable for rumen microbiome analysis - references?
</p></div></div>

<div class="footdef"><sup><a id="fn.5" class="footnum" href="#fnr.5">5</a></sup> <div class="footpara"><p class="footpara">
P-value to back this up?
</p></div></div>

<div class="footdef"><sup><a id="fn.6" class="footnum" href="#fnr.6">6</a></sup> <div class="footpara"><p class="footpara">
For sugarcane bagasse, the rate appears constant to me
</p></div></div>

<div class="footdef"><sup><a id="fn.7" class="footnum" href="#fnr.7">7</a></sup> <div class="footpara"><p class="footpara">
Maximum for lucerne hay is less than 48 h. More like 32 h
</p></div></div>

<div class="footdef"><sup><a id="fn.8" class="footnum" href="#fnr.8">8</a></sup> <div class="footpara"><p class="footpara">
This should be section 3.3
</p></div></div>

<div class="footdef"><sup><a id="fn.9" class="footnum" href="#fnr.9">9</a></sup> <div class="footpara"><p class="footpara">
Should this be Observed OTUs?
</p></div></div>

<div class="footdef"><sup><a id="fn.10" class="footnum" href="#fnr.10">10</a></sup> <div class="footpara"><p class="footpara">
Data not shown? Why? This might be a good comparison
</p></div></div>

<div class="footdef"><sup><a id="fn.11" class="footnum" href="#fnr.11">11</a></sup> <div class="footpara"><p class="footpara">
New subheading? Beta diversity analysis
</p></div></div>

<div class="footdef"><sup><a id="fn.12" class="footnum" href="#fnr.12">12</a></sup> <div class="footpara"><p class="footpara">
I'm not sure how to interprete this figure
</p></div></div>

<div class="footdef"><sup><a id="fn.13" class="footnum" href="#fnr.13">13</a></sup> <div class="footpara"><p class="footpara">
I don't think this paragraph is necessary as the focus should be on genus/species-level analysis
</p></div></div>

<div class="footdef"><sup><a id="fn.14" class="footnum" href="#fnr.14">14</a></sup> <div class="footpara"><p class="footpara">
Perhaps just include the genera that significantly changed in this figure?
</p></div></div>

<div class="footdef"><sup><a id="fn.15" class="footnum" href="#fnr.15">15</a></sup> <div class="footpara"><p class="footpara">
Don't need to show this P value
</p></div></div>

<div class="footdef"><sup><a id="fn.16" class="footnum" href="#fnr.16">16</a></sup> <div class="footpara"><p class="footpara">
Greater than 0.0 what?
</p></div></div>

<div class="footdef"><sup><a id="fn.17" class="footnum" href="#fnr.17">17</a></sup> <div class="footpara"><p class="footpara">
Less than what?
</p></div></div>

<div class="footdef"><sup><a id="fn.18" class="footnum" href="#fnr.18">18</a></sup> <div class="footpara"><p class="footpara">
Is it possible to just contrast these findings with the bagasse findings and highlight differences? It all sounds very repetitive the way it is currently written
</p></div></div>

<div class="footdef"><sup><a id="fn.19" class="footnum" href="#fnr.19">19</a></sup> <div class="footpara"><p class="footpara">
How is this section different to the previous one? Can the important comparisons be incorporated into one section? Primarily focussing on species/genus-level analysis
</p></div></div>

<div class="footdef"><sup><a id="fn.20" class="footnum" href="#fnr.20">20</a></sup> <div class="footpara"><p class="footpara">
Is there a figure highlighting these differences?
</p></div></div>

<div class="footdef"><sup><a id="fn.21" class="footnum" href="#fnr.21">21</a></sup> <div class="footpara"><p class="footpara">
Again, this is all quite repetitive. I wonder if this and the previous two sections can be condensed into one, highlighting only the major differences? This will help keep the reader engaged
</p></div></div>

<div class="footdef"><sup><a id="fn.22" class="footnum" href="#fnr.22">22</a></sup> <div class="footpara"><p class="footpara">
This section be incorporated into the "Epiphytic bacterial communities on sugarcane bagasse and Lucerne hay" section if you change the title of that section
</p></div></div>

<div class="footdef"><sup><a id="fn.23" class="footnum" href="#fnr.23">23</a></sup> <div class="footpara"><p class="footpara">
This sounds like it should go in the Introduction
</p></div></div>

<div class="footdef"><sup><a id="fn.24" class="footnum" href="#fnr.24">24</a></sup> <div class="footpara"><p class="footpara">
This sounds like the start of the Discussion
</p></div></div>

<div class="footdef"><sup><a id="fn.25" class="footnum" href="#fnr.25">25</a></sup> <div class="footpara"><p class="footpara">
Again, this sounds like background information and may be better placed in the Introduction
</p></div></div>

<div class="footdef"><sup><a id="fn.26" class="footnum" href="#fnr.26">26</a></sup> <div class="footpara"><p class="footpara">
No need to go into so much detail here as that was provided in the results section. Just make a quick reference to the fact that there were big differences in digestibility
</p></div></div>

<div class="footdef"><sup><a id="fn.27" class="footnum" href="#fnr.27">27</a></sup> <div class="footpara"><p class="footpara">
Extra timepoints analysed in this study provided novel insight into phases of digestion that had not previously been observed. This is the key takeaway point and I'm not sure the detail provided before this is necessary to include. Just point the reader to the relevant references
</p></div></div>

<div class="footdef"><sup><a id="fn.28" class="footnum" href="#fnr.28">28</a></sup> <div class="footpara"><p class="footpara">
This is the key point and just use appropriate references to confirm that your findings are similar to previous ones. No need for all the detail
</p></div></div>

<div class="footdef"><sup><a id="fn.29" class="footnum" href="#fnr.29">29</a></sup> <div class="footpara"><p class="footpara">
This paragraph basically says that your findings are in agreement with what is already known. That's fine. But is there anything about your particular dataset that is novel? E.g. any novel species-level observation that made use of long Pacbio reads?
</p></div></div>

<div class="footdef"><sup><a id="fn.30" class="footnum" href="#fnr.30">30</a></sup> <div class="footpara"><p class="footpara">
Again, you are highlighting similarities with other studies here. Are there any differences/novel insights?
</p></div></div>

<div class="footdef"><sup><a id="fn.31" class="footnum" href="#fnr.31">31</a></sup> <div class="footpara"><p class="footpara">
I dont really follow this
</p></div></div>

<div class="footdef"><sup><a id="fn.32" class="footnum" href="#fnr.32">32</a></sup> <div class="footpara"><p class="footpara">
Perhaps start a new subsection for lucerne hay. Likewise, make a new sub-section for the sugarcane bagasse section above
</p></div></div>

<div class="footdef"><sup><a id="fn.33" class="footnum" href="#fnr.33">33</a></sup> <div class="footpara"><p class="footpara">
More similarities. Any differences?
</p></div></div>

<div class="footdef"><sup><a id="fn.34" class="footnum" href="#fnr.34">34</a></sup> <div class="footpara"><p class="footpara">
Perhaps this can go closer to the start of the Discussion as it summarises the key findings
</p></div></div>

<div class="footdef"><sup><a id="fn.35" class="footnum" href="#fnr.35">35</a></sup> <div class="footpara"><p class="footpara">
The summary should focus on how your approach led to novel insights and how these in turn might have practical applications. You should probably also address limitations such as the lack of analysis of fungi etc. As otherwise this will be picked up by the reviewers
</p></div></div>

<div class="footdef"><sup><a id="fn.36" class="footnum" href="#fnr.36">36</a></sup> <div class="footpara"><p class="footpara">
The benefits of which were?
</p></div></div>

<div class="footdef"><sup><a id="fn.37" class="footnum" href="#fnr.37">37</a></sup> <div class="footpara"><p class="footpara">
CARF, UQ Gatton etc
</p></div></div>

 --></div>
<div id="postamble" class="status">
<p class="author">Author: Kevin Dudley</p>
<p class="date">Created: 2022-03-07 Mon 16:36</p>
<p class="validation"><a href="https://validator.w3.org/check?uri=referer">Validate</a></p>
</div>
</body>
</html>
